<!doctype html><html lang=en-us><head><link rel=preconnect href=https://fonts.googleapis.com><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@700&display=swap" rel=stylesheet><meta charset=utf-8><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><script src=https://ajax.googleapis.com/ajax/libs/jquery/3.6.1/jquery.min.js></script><script>var unis={wat:true,sas:true};$(document).on("change","input[type='checkbox']",function(){if(this.id.startsWith("check_year")){const collection=document.getElementById("main").children;for(child of collection)
{if(child.id.startsWith(this.id.slice(-9)))
{this.checked?child.style.display="initial":child.style.display="none";}}}else if(this.id.startsWith("uni")){var uni=this.id.slice(-3);this.checked?(unis[uni]=true):(unis[uni]=false);const collection=document.querySelectorAll("[data-uni="+uni+"]");for(child of collection){if(this.checked){child.parentNode.parentNode.setAttribute("style","display:flex !important")}else{disp_off_flag=true;for(child of child.parentNode.querySelectorAll("[data-uni]")){if(unis[child.getAttribute("data-uni")]==true){disp_off_flag=false;break;}}
if(disp_off_flag){child.parentNode.parentNode.setAttribute("style","display:none !important");}}}}else if(this.id.startsWith("prj")){var prj=this.id.slice(-4);const collection=document.querySelectorAll("[data-prj="+prj+"]");for(child of collection){(this.checked)?child.setAttribute("style","display:initial !important"):child.setAttribute("style","display:none !important")}}});</script><!--[if IEMobile]><meta http-equiv=cleartype content="on"><![endif]--><title>Robert Delatolla - GWF Publications</title><meta name=generator content="Hugo 0.68.3"><link href=/gwf-publications/gwficon.ico rel="shortcut icon" type=image/x-icon><link rel=stylesheet href=/gwf-publications/css/main.min.8976777c0832d068a49d330764e507857027f1efa3b8501cf349b0e2db7410fc.css media=screen><link rel=stylesheet href=https://use.fontawesome.com/releases/v5.7.2/css/all.css integrity=sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr crossorigin=anonymous><link rel=stylesheet href=/gwf-publications/css/academicons.min.css></head><body><header style="z-index:1002;top:0;position:sticky;display:block;box-sizing:border-box;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;line-height:1.5;color:#222;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%"><div id=navbar-container style="height:54px;width:100%;background:rgba(255,255,255,.95);overflow:hidden;-webkit-box-shadow:rgba(0,0,0,.09)0 1px 0;box-shadow:rgba(0,0,0,.09)0 1px 0;box-sizing:border-box;display:block;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;line-height:1.5;color:#222;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%"><a href=/gwf-publications/ style="color:#0072bc;text-decoration:none;display:inline-block;height:100%;line-height:54px;border-right:1px solid #e6e6e6;margin-left:6px;padding-right:6px;outline-offset:-4px!important;font-weight:700;background:0 0;box-sizing:border-box;cursor:pointer;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;-webkit-tap-highlight-color:transparent"><img src=/gwf-publications/images/gwf-logo.png alt="ACL Logo" style="margin-right:0;height:54px;padding:6px;width:auto;margin-top:-2px;max-width:100%;vertical-align:middle;border:0;box-sizing:border-box;color:#0072bc;line-height:54px;font-weight:700;cursor:pointer;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%"></a>
<a href=/gwf-publications/ style="text-decoration:none;color:#222!important;display:inline-block;font-size:16px;line-height:54px;font-weight:700;outline-offset:-4px!important;padding:0 6px;margin:0 -6px;background:0 0;box-sizing:border-box;cursor:pointer;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%;padding-left:4px">Global Water Futures</a></div></header><div id=main-container class=container><section id=main><h2 id=title><span class=font-weight-normal>Robert</span> <span class=font-weight-bold>Delatolla</span></h2><p class="font-weight-light text-muted">wat</p><hr><div class=row><div class=col-lg-9><h4>2022</h4><div data-prj=prj1><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://gwf-uwaterloo.github.io/gwf-publications/G22-18001.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G22-18001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G22-18001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G22-18001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G22-18001/>Community Surveillance of Omicron in Ontario: Wastewater-based Epidemiology Comes of Age.</a></strong><br><a href=/gwf-publications/people/A/Authors-presented-in-alphabetical-order/ data-uni=wat>Authors presented in alphabetical order:</a>,
<a href=/gwf-publications/people/E/Eric-Arts/ data-uni=wat>Eric Arts</a>,
<a href=/gwf-publications/people/S/Stephen-Brown/ data-uni=wat>Stephen Brown</a>,
<a href=/gwf-publications/people/D/David-Bulir/ data-uni=wat>David Bulir</a>,
<a href=/gwf-publications/people/T/Trevor-C-Charles/ data-uni=wat>Trevor C. Charles</a>,
<a href=/gwf-publications/people/C/Christopher-T-DeGroot/ data-uni=wat>Christopher T. DeGroot</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=wat>Robert Delatolla</a>,
<a href=/gwf-publications/people/J/Jean-Paul-Desaulniers/ data-uni=wat>Jean-Paul Desaulniers</a>,
<a href=/gwf-publications/people/E/Elizabeth-A-Edwards/ data-uni=wat>Elizabeth A. Edwards</a>,
<a href=/gwf-publications/people/M/Meghan-Fuzzen/ data-uni=wat>Meghan Fuzzen</a>,
<a href=/gwf-publications/people/K/Kimberley-Gilbride/ data-uni=wat>Kimberley Gilbride</a>,
<a href=/gwf-publications/people/J/Jodi-Gilchrist/ data-uni=wat>Jodi Gilchrist</a>,
<a href=/gwf-publications/people/L/Lawrence-Goodridge/ data-uni=wat>Lawrence Goodridge</a>,
<a href=/gwf-publications/people/T/Tyson-E-Graber/ data-uni=wat>Tyson E. Graber</a>,
<a href=/gwf-publications/people/M/Marc-Habash/ data-uni=wat>Marc Habash</a>,
<a href=/gwf-publications/people/P/Peter-J%C3%BCni/ data-uni=wat>Peter Jüni</a>,
<a href=/gwf-publications/people/A/Andrea-Kirkwood/ data-uni=wat>Andrea Kirkwood</a>,
<a href=/gwf-publications/people/J/James-Knockleby/ data-uni=wat>James Knockleby</a>,
<a href=/gwf-publications/people/C/Christopher-Kyle/ data-uni=wat>Christopher Kyle</a>,
<a href=/gwf-publications/people/C/Chrystal-Landgraff/ data-uni=wat>Chrystal Landgraff</a>,
<a href=/gwf-publications/people/C/Chand-Mangat/ data-uni=wat>Chand Mangat</a>,
<a href=/gwf-publications/people/D/Douglas-G-Manuel/ data-uni=wat>Douglas G. Manuel</a>,
<a href=/gwf-publications/people/R/R-Michael-McKay/ data-uni=wat>R. Michael McKay</a>,
<a href=/gwf-publications/people/E/Edgard-Mejia/ data-uni=wat>Edgard Mejia</a>,
<a href=/gwf-publications/people/A/Aleksandra-Mloszewska/ data-uni=wat>Aleksandra Mloszewska</a>,
<a href=/gwf-publications/people/B/Banu-Ormeci/ data-uni=wat>Banu Ormeci</a>,
<a href=/gwf-publications/people/C/Claire-Oswald/ data-uni=wat>Claire Oswald</a>,
<a href=/gwf-publications/people/S/Sarah-Jane-Payne/ data-uni=wat>Sarah Jane Payne</a>,
<a href=/gwf-publications/people/H/Hui-Peng/ data-uni=wat>Hui Peng</a>,
<a href=/gwf-publications/people/S/Shelley-Peterson/ data-uni=wat>Shelley Peterson</a>,
<a href=/gwf-publications/people/A/Art-FY-Poon/ data-uni=wat>Art F.Y. Poon</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=wat>Mark R. Servos</a>,
<a href=/gwf-publications/people/D/Denina-Simmons/ data-uni=wat>Denina Simmons</a>,
<a href=/gwf-publications/people/J/Jianxian-Sun/ data-uni=wat>Jianxian Sun</a>,
<a href=/gwf-publications/people/M/Minqing-Yang/ data-uni=wat>Minqing Yang</a>,
<a href=/gwf-publications/people/G/Gustavo-Ybazeta/ data-uni=wat>Gustavo Ybazeta</a><br><a href=/gwf-publications/volumes/G22-18/ class=text-muted></a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G22-18001><div class="card-body p-3 small">Abstract Wastewater-based surveillance of SARS-CoV-2 RNA has been implemented at building, neighbourhood, and city levels throughout the world. Implementation strategies and analysis methods differ, but they all aim to provide rapid and reliable information about community COVID-19 health states. A viable and sustainable SARS-CoV-2 surveillance network must not only provide reliable and timely information about COVID-19 trends, but also provide for scalability as well as accurate detection of known or unknown emerging variants. Emergence of the SARS-CoV-2 variant of concern Omicron in late Fall 2021 presented an excellent opportunity to benchmark individual and aggregated data outputs of the Ontario Wastewater Surveillance Initiative in Canada; this public health-integrated surveillance network monitors wastewaters from over 10 million people across major population centres of the province. We demonstrate that this coordinated approach provides excellent situational awareness, comparing favourably with traditional clinical surveillance measures. Thus, aggregated datasets compiled from multiple wastewater-based surveillance nodes can provide sufficient sensitivity (i.e., early indication of increasing and decreasing incidence of SARS-CoV-2) and specificity (i.e., allele frequency estimation of emerging variants) with which to make informed public health decisions at regional- and state-levels.</div></div><div data-prj=prj1><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://gwf-uwaterloo.github.io/gwf-publications/G22-18002.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G22-18002.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G22-18002 data-toggle=collapse aria-expanded=false aria-controls=abstract-G22-18002 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G22-18002/>Multiplex RT-qPCR assay (N200) to detect and estimate prevalence of multiple SARS-CoV-2 Variants of Concern in wastewater</a></strong><br><a href=/gwf-publications/people/M/Meghan-Fuzzen/ data-uni=wat>Meghan Fuzzen</a>,
<a href=/gwf-publications/people/N/Nathanael-BJ-Harper/ data-uni=wat>Nathanael B.J. Harper</a>,
<a href=/gwf-publications/people/H/Hadi-A-Dhiyebi/ data-uni=wat>Hadi A. Dhiyebi</a>,
<a href=/gwf-publications/people/N/Nivetha-Srikanthan/ data-uni=wat>Nivetha Srikanthan</a>,
<a href=/gwf-publications/people/S/Samina-Hayat/ data-uni=wat>Samina Hayat</a>,
<a href=/gwf-publications/people/S/Shelley-W-Peterson/ data-uni=wat>Shelley W. Peterson</a>,
<a href=/gwf-publications/people/I/Ivy-Yang/ data-uni=wat>Ivy Yang</a>,
<a href=/gwf-publications/people/J/JX-Sun/ data-uni=wat>J.X. Sun</a>,
<a href=/gwf-publications/people/E/Elizabeth-A-Edwards/ data-uni=wat>Elizabeth A. Edwards</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=wat>John P. Giesy</a>,
<a href=/gwf-publications/people/C/Chand-S-Mangat/ data-uni=wat>Chand S. Mangat</a>,
<a href=/gwf-publications/people/T/Tyson-E-Graber/ data-uni=wat>Tyson E. Graber</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=wat>Robert Delatolla</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=wat>Mark R. Servos</a><br><a href=/gwf-publications/volumes/G22-18/ class=text-muted></a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G22-18002><div class="card-body p-3 small">Abstract Wastewater-based surveillance (WBS) has become an effective tool around the globe for indirect monitoring of COVID-19 in communities. Quantities of viral fragments of SARS-CoV-2 in wastewater are related to numbers of clinical cases of COVID-19 reported within the corresponding sewershed. Variants of Concern (VOCs) have been detected in wastewater by use of reverse transcription quantitative polymerase chain reaction (RT-qPCR) or sequencing. A multiplex RT-qPCR assay to detect and estimate the prevalence of multiple VOCs, including Omicron/Alpha, Beta, Gamma, and Delta, in wastewater RNA extracts was developed and validated. The probe-based multiplex assay, named “N200” focuses on amino acids 199-202, a region of the N gene that contains several mutations that are associated with variants of SARS- CoV-2 within a single amplicon. Each of the probes in the N200 assay are specific to the targeted mutations and worked equally well in single- and multi-plex modes. To estimate prevalence of each VOC, the abundance of the targeted mutation was compared with a non- mutated region within the same amplified region. The N200 assay was applied to monitor frequencies of VOCs in wastewater extracts from six sewersheds in Ontario, Canada collected between December 1, 2021, and January 4, 2022. Using the N200 assay, the replacement of the Delta variant along with the introduction and rapid dominance of the Omicron variant were monitored in near real-time, as they occurred nearly simultaneously at all six locations. The N200 assay is robust and efficient for wastewater surveillance can be adopted into VOC monitoring programs or replace more laborious assays currently being used to monitor SARS- CoV-2 and its VOCs.</div></div><h4>2021</h4><div data-prj=prj2><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://gwf-uwaterloo.github.io/gwf-publications/G21-41001.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-41001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-41001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-41001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-41001/>Comparison of approaches to quantify SARS-CoV-2 in wastewater using RT-qPCR: Results and implications from a collaborative inter-laboratory study in Canada</a></strong><br><a href=/gwf-publications/people/A/Alex-HS-Chik/ data-uni=wat>Alex H.S. Chik</a>,
<a href=/gwf-publications/people/M/Melissa-B-Glier/ data-uni=wat>Melissa B. Glier</a>,
<a href=/gwf-publications/people/M/Mark-Servos/ data-uni=wat>Mark Servos</a>,
<a href=/gwf-publications/people/C/Chand-S-Mangat/ data-uni=wat>Chand S. Mangat</a>,
<a href=/gwf-publications/people/X/Xiao-Li-Pang/ data-uni=wat>Xiao-Li Pang</a>,
<a href=/gwf-publications/people/Y/Yuanyuan-Qiu/ data-uni=wat>Yuanyuan Qiu</a>,
<a href=/gwf-publications/people/P/Patrick-M-DAoust/ data-uni=wat>Patrick M. D'Aoust</a>,
<a href=/gwf-publications/people/J/Jean-Baptiste-Burnet/ data-uni=wat>Jean-Baptiste Burnet</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=wat>Robert Delatolla</a>,
<a href=/gwf-publications/people/S/Sarah-Dorner/ data-uni=wat>Sarah Dorner</a>,
<a href=/gwf-publications/people/Q/Qiudi-Geng/ data-uni=wat>Qiudi Geng</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=wat>John P. Giesy</a>,
<a href=/gwf-publications/people/R/Robert-Mike-McKay/ data-uni=wat>Robert Mike McKay</a>,
<a href=/gwf-publications/people/M/Michael-R-Mulvey/ data-uni=wat>Michael R. Mulvey</a>,
<a href=/gwf-publications/people/N/Natalie-Prystajecky/ data-uni=wat>Natalie Prystajecky</a>,
<a href=/gwf-publications/people/N/Nivetha-Srikanthan/ data-uni=wat>Nivetha Srikanthan</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=wat>Yuwei Xie</a>,
<a href=/gwf-publications/people/B/Bernadette-Conant/ data-uni=wat>Bernadette Conant</a>,
<a href=/gwf-publications/people/S/Steve-E-Hrudey/ data-uni=wat>Steve E. Hrudey</a><br><a href=/gwf-publications/volumes/G21-41/ class=text-muted>Journal of Environmental Sciences, Volume 107</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-41001><div class="card-body p-3 small">Detection of SARS-CoV-2 RNA in wastewater is a promising tool for informing public health decisions during the COVID-19 pandemic. However, approaches for its analysis by use of reverse transcription quantitative polymerase chain reaction (RT-qPCR) are still far from standardized globally. To characterize inter- and intra-laboratory variability among results when using various methods deployed across Canada, aliquots from a real wastewater sample were spiked with surrogates of SARS-CoV-2 (gamma-radiation inactivated SARS-CoV-2 and human coronavirus strain 229E [HCoV-229E]) at low and high levels then provided "blind" to eight laboratories. Concentration estimates reported by individual laboratories were consistently within a 1.0-log10 range for aliquots of the same spiked condition. All laboratories distinguished between low- and high-spikes for both surrogates. As expected, greater variability was observed in the results amongst laboratories than within individual laboratories, but SARS-CoV-2 RNA concentration estimates for each spiked condition remained mostly within 1.0-log10 ranges. The no-spike wastewater aliquots provided yielded non-detects or trace levels (&lt;20 gene copies/mL) of SARS-CoV-2 RNA. Detections appear linked to methods that included or focused on the solids fraction of the wastewater matrix and might represent in-situ SARS-CoV-2 to the wastewater sample. HCoV-229E RNA was not detected in the no-spike aliquots. Overall, all methods yielded comparable results at the conditions tested. Partitioning behavior of SARS-CoV-2 and spiked surrogates in wastewater should be considered to evaluate method effectiveness. A consistent method and laboratory to explore wastewater SARS-CoV-2 temporal trends for a given system, with appropriate quality control protocols and documented in adequate detail should succeed.</div></div><div data-prj=prj1><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://gwf-uwaterloo.github.io/gwf-publications/G21-47001.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-62010.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-62010 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-62010 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-62010/>Near real-time determination of B.1.1.7 in proportion to total SARS-CoV-2 viral load in wastewater using an allele-specific primer extension PCR strategy</a></strong><br><a href=/gwf-publications/people/T/Tyson-E-Graber/ data-uni=wat>Tyson E. Graber</a>,
<a href=/gwf-publications/people/%C3%89/%C3%89lisabeth-Mercier/ data-uni=wat>Élisabeth Mercier</a>,
<a href=/gwf-publications/people/K/Kamya-Bhatnagar/ data-uni=wat>Kamya Bhatnagar</a>,
<a href=/gwf-publications/people/M/Meghan-Fuzzen/ data-uni=wat>Meghan Fuzzen</a>,
<a href=/gwf-publications/people/P/Patrick-M-DAoust/ data-uni=wat>Patrick M. D’Aoust</a>,
<a href=/gwf-publications/people/H/Huy-Dung-Hoang/ data-uni=wat>Huy-Dung Hoang</a>,
<a href=/gwf-publications/people/X/Xin-Tian/ data-uni=wat>Xin Tian</a>,
<a href=/gwf-publications/people/S/Syeda-Tasneem-Towhid/ data-uni=wat>Syeda Tasneem Towhid</a>,
<a href=/gwf-publications/people/J/Julio-Plaza-Diaz/ data-uni=wat>Julio Plaza Diaz</a>,
<a href=/gwf-publications/people/T/Tommy-Alain/ data-uni=wat>Tommy Alain</a>,
<a href=/gwf-publications/people/A/Ainslie-Butler/ data-uni=wat>Ainslie Butler</a>,
<a href=/gwf-publications/people/L/Lawrence-Goodridge/ data-uni=wat>Lawrence Goodridge</a>,
<a href=/gwf-publications/people/M/Mark-Servos/ data-uni=wat>Mark Servos</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=wat>Robert Delatolla</a><br><a href=/gwf-publications/volumes/G21-62/ class=text-muted></a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-62010><div class="card-body p-3 small">Abstract The coronavirus disease 2019 (COVID-19) pandemic caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has claimed millions of lives to date. Antigenic drift has resulted in viral variants with putatively greater transmissibility, virulence, or both. Early and near real-time detection of these variants of concern (VOC) and the ability to accurately follow their incidence and prevalence in communities is wanting. Wastewater-based epidemiology (WBE), which uses nucleic acid amplification tests to detect viral fragments, is a faithful proxy of COVID-19 incidence and prevalence, and thus offers the potential to monitor VOC viral load in a given population. Here, we describe and validate a primer extension PCR strategy targeting a signature mutation in the N gene of SARS-CoV-2. This allows quantification of the proportional expression of B.1.1.7 versus non-B.1.1.7 alleles in wastewater without the need to employ quantitative RT-PCR standard curves. We show that the wastewater B.1.1.7 profile correlates with its clinical counterpart and benefits from a near real-time and facile data collection and reporting pipeline. This assay can be quickly implemented within a current SARS-CoV-2 WBE framework with minimal cost; allowing early and contemporaneous estimates of B.1.1.7 community transmission prior to, or in lieu of, clinical screening and identification. Our study demonstrates that this strategy can provide public health units with an additional and much needed tool to rapidly triangulate VOC incidence/prevalence with high sensitivity and lineage specificity.</div></div><div data-prj=prj2><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://gwf-uwaterloo.github.io/gwf-publications/G21-34001.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-222001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-222001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-222001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-222001/>Quantitative analysis of SARS-CoV-2 RNA from wastewater solids in communities with low COVID-19 incidence and prevalence</a></strong><br><a href=/gwf-publications/people/P/Patrick-M-DAoust/ data-uni=wat>Patrick M. D'Aoust</a>,
<a href=/gwf-publications/people/E/Elisabeth-Mercier/ data-uni=wat>Elisabeth Mercier</a>,
<a href=/gwf-publications/people/D/Danika-Montpetit/ data-uni=wat>Danika Montpetit</a>,
<a href=/gwf-publications/people/J/Jian-Jun-Jia/ data-uni=wat>Jian-Jun Jia</a>,
<a href=/gwf-publications/people/I/Ilya-Alexandrov/ data-uni=wat>Ilya Alexandrov</a>,
<a href=/gwf-publications/people/N/Nafisa-Neault/ data-uni=wat>Nafisa Neault</a>,
<a href=/gwf-publications/people/A/Aiman-Tariq-Baig/ data-uni=wat>Aiman Tariq Baig</a>,
<a href=/gwf-publications/people/J/Janice-Mayne/ data-uni=wat>Janice Mayne</a>,
<a href=/gwf-publications/people/X/Xu-Zhang/ data-uni=wat>Xu Zhang</a>,
<a href=/gwf-publications/people/T/Tommy-Alain/ data-uni=wat>Tommy Alain</a>,
<a href=/gwf-publications/people/M/Marc-Andr%C3%A9-Langlois/ data-uni=wat>Marc-André Langlois</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=wat>Mark R. Servos</a>,
<a href=/gwf-publications/people/M/Malcolm-MacKenzie/ data-uni=wat>Malcolm MacKenzie</a>,
<a href=/gwf-publications/people/D/Daniel-Figeys/ data-uni=wat>Daniel Figeys</a>,
<a href=/gwf-publications/people/A/Alex-E-MacKenzie/ data-uni=wat>Alex E. MacKenzie</a>,
<a href=/gwf-publications/people/T/Tyson-E-Graber/ data-uni=wat>Tyson E. Graber</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=wat>Robert Delatolla</a><br><a href=/gwf-publications/volumes/G21-222/ class=text-muted>Water Research, Volume 188</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-222001><div class="card-body p-3 small">• RT-ddPCR is more sensitive to inhibitors than RT-qPCR for primary clarified sludge. • Primary clarified sludge has elevated frequency of SARS-CoV-2 RNA detection. • Primary clarified sludge allows detection of RNA during low COVID-19 incidence. • PMMoV normalization of RNA data reduces noise and increases precision. • PMMoV normalization of RNA shows strongest correlation to epidemiological metrics. In the absence of an effective vaccine to prevent COVID-19 it is important to be able to track community infections to inform public health interventions aimed at reducing the spread and therefore reduce pressures on health-care, improve health outcomes and reduce economic uncertainty. Wastewater surveillance has rapidly emerged as a potential tool to effectively monitor community infections through measuring trends of RNA signal in wastewater systems. In this study SARS-CoV-2 viral RNA N1 and N2 gene regions are quantified in solids collected from influent post grit solids (PGS) and primary clarified sludge (PCS) in two water resource recovery facilities (WRRF) serving Canada's national capital region, i.e., the City of Ottawa, ON (pop. ≈ 1.1M) and the City of Gatineau, QC (pop. ≈ 280K). PCS samples show signal inhibition using RT-ddPCR compared to RT-qPCR, with PGS samples showing similar quantifiable concentrations of RNA using both assays. RT-qPCR shows higher frequency of detection of N1 and N2 gene regions in PCS (92.7, 90.6%, n = 6) as compared to PGS samples (79.2, 82.3%, n = 5). Sampling of PCS may therefore be an effective approach for SARS-CoV-2 viral quantification, especially during periods of declining and low COVID-19 incidence in the community. The pepper mild mottle virus (PMMoV) is determined to have a less variable RNA signal in PCS over a three month period for two WRRFs, regardless of environmental conditions, compared to Bacteroides 16S rRNA or human 18S rRNA, making PMMoV a potentially useful biomarker for normalization of SARS-CoV-2 signal. PMMoV-normalized PCS RNA signal from WRRFs of two cities correlated with the regional public health epidemiological metrics, identifying PCS normalized to a fecal indicator (PMMoV) as a potentially effective tool for monitoring trends during decreasing and low-incidence of infection of SARS-Cov-2 in communities.</div></div></div><div class=col-lg-3><a class="btn btn-lg btn-secondary btn-block mb-2" href="https://www.semanticscholar.org/search?q=Robert+Delatolla" title="Search for 'Robert Delatolla' on Semantic Scholar"><i class="ai ai-semantic-scholar"></i><span class=pl-sm-2>Search</span></a><div class=row><div class="col-12 col-md-6 col-lg-12"><div class=card><h5 class=card-header>Co-authors</h5><ul class="list-group list-group-flush list-group-compact"><li class=list-group-item><a href=/gwf-publications/people/T/Tyson-E-Graber/ class=align-middle data-uni=wat>Tyson E. Graber</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Meghan-Fuzzen/ class=align-middle data-uni=wat>Meghan Fuzzen</a>
<span class="badge badge-secondary align-middle ml-2">3</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Mark-R-Servos/ class=align-middle data-uni=wat>Mark R. Servos</a>
<span class="badge badge-secondary align-middle ml-2">3</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Mark-Servos/ class=align-middle data-uni=wat>Mark Servos</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Chand-S-Mangat/ class=align-middle data-uni=wat>Chand S. Mangat</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class="list-group-item list-group-toggle-btn py-1" data-toggle=collapse data-target=#more-coauthors aria-expanded=false aria-controls=more-coauthors>show all...</li><div class="collapse border-top" id=more-coauthors><li class=list-group-item><a href=/gwf-publications/people/P/Patrick-M-DAoust/ class=align-middle data-uni=wat>Patrick M. D'Aoust</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/J/John-P-Giesy/ class=align-middle data-uni=wat>John P. Giesy</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Nivetha-Srikanthan/ class=align-middle data-uni=wat>Nivetha Srikanthan</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/T/Tommy-Alain/ class=align-middle data-uni=wat>Tommy Alain</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/L/Lawrence-Goodridge/ class=align-middle data-uni=wat>Lawrence Goodridge</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/E/Elizabeth-A-Edwards/ class=align-middle data-uni=wat>Elizabeth A. Edwards</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Alex-HS-Chik/ class=align-middle data-uni=wat>Alex H.S. Chik</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Melissa-B-Glier/ class=align-middle data-uni=wat>Melissa B. Glier</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/X/Xiao-Li-Pang/ class=align-middle data-uni=wat>Xiao-Li Pang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yuanyuan-Qiu/ class=align-middle data-uni=wat>Yuanyuan Qiu</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jean-Baptiste-Burnet/ class=align-middle data-uni=wat>Jean-Baptiste Burnet</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Sarah-Dorner/ class=align-middle data-uni=wat>Sarah Dorner</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Q/Qiudi-Geng/ class=align-middle data-uni=wat>Qiudi Geng</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/R/Robert-Mike-McKay/ class=align-middle data-uni=wat>Robert Mike McKay</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Michael-R-Mulvey/ class=align-middle data-uni=wat>Michael R. Mulvey</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Natalie-Prystajecky/ class=align-middle data-uni=wat>Natalie Prystajecky</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yuwei-Xie/ class=align-middle data-uni=wat>Yuwei Xie</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/B/Bernadette-Conant/ class=align-middle data-uni=wat>Bernadette Conant</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Steve-E-Hrudey/ class=align-middle data-uni=wat>Steve E. Hrudey</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/%C3%89/%C3%89lisabeth-Mercier/ class=align-middle data-uni=wat>Élisabeth Mercier</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/K/Kamya-Bhatnagar/ class=align-middle data-uni=wat>Kamya Bhatnagar</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/P/Patrick-M-DAoust/ class=align-middle data-uni=wat>Patrick M. D’Aoust</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/H/Huy-Dung-Hoang/ class=align-middle data-uni=wat>Huy-Dung Hoang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/X/Xin-Tian/ class=align-middle data-uni=wat>Xin Tian</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Syeda-Tasneem-Towhid/ class=align-middle data-uni=wat>Syeda Tasneem Towhid</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Julio-Plaza-Diaz/ class=align-middle data-uni=wat>Julio Plaza Diaz</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Ainslie-Butler/ class=align-middle data-uni=wat>Ainslie Butler</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/E/Elisabeth-Mercier/ class=align-middle data-uni=wat>Elisabeth Mercier</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/D/Danika-Montpetit/ class=align-middle data-uni=wat>Danika Montpetit</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jian-Jun-Jia/ class=align-middle data-uni=wat>Jian-Jun Jia</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/I/Ilya-Alexandrov/ class=align-middle data-uni=wat>Ilya Alexandrov</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Nafisa-Neault/ class=align-middle data-uni=wat>Nafisa Neault</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Aiman-Tariq-Baig/ class=align-middle data-uni=wat>Aiman Tariq Baig</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Janice-Mayne/ class=align-middle data-uni=wat>Janice Mayne</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/X/Xu-Zhang/ class=align-middle data-uni=wat>Xu Zhang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Marc-Andr%C3%A9-Langlois/ class=align-middle data-uni=wat>Marc-André Langlois</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Malcolm-MacKenzie/ class=align-middle data-uni=wat>Malcolm MacKenzie</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/D/Daniel-Figeys/ class=align-middle data-uni=wat>Daniel Figeys</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Alex-E-MacKenzie/ class=align-middle data-uni=wat>Alex E. MacKenzie</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Authors-presented-in-alphabetical-order/ class=align-middle data-uni=wat>Authors presented in alphabetical order:</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/E/Eric-Arts/ class=align-middle data-uni=wat>Eric Arts</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Stephen-Brown/ class=align-middle data-uni=wat>Stephen Brown</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/D/David-Bulir/ class=align-middle data-uni=wat>David Bulir</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/T/Trevor-C-Charles/ class=align-middle data-uni=wat>Trevor C. Charles</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Christopher-T-DeGroot/ class=align-middle data-uni=wat>Christopher T. DeGroot</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jean-Paul-Desaulniers/ class=align-middle data-uni=wat>Jean-Paul Desaulniers</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/K/Kimberley-Gilbride/ class=align-middle data-uni=wat>Kimberley Gilbride</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jodi-Gilchrist/ class=align-middle data-uni=wat>Jodi Gilchrist</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Marc-Habash/ class=align-middle data-uni=wat>Marc Habash</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/P/Peter-J%C3%BCni/ class=align-middle data-uni=wat>Peter Jüni</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Andrea-Kirkwood/ class=align-middle data-uni=wat>Andrea Kirkwood</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/James-Knockleby/ class=align-middle data-uni=wat>James Knockleby</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Christopher-Kyle/ class=align-middle data-uni=wat>Christopher Kyle</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Chrystal-Landgraff/ class=align-middle data-uni=wat>Chrystal Landgraff</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Chand-Mangat/ class=align-middle data-uni=wat>Chand Mangat</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/D/Douglas-G-Manuel/ class=align-middle data-uni=wat>Douglas G. Manuel</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/R/R-Michael-McKay/ class=align-middle data-uni=wat>R. Michael McKay</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/E/Edgard-Mejia/ class=align-middle data-uni=wat>Edgard Mejia</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Aleksandra-Mloszewska/ class=align-middle data-uni=wat>Aleksandra Mloszewska</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/B/Banu-Ormeci/ class=align-middle data-uni=wat>Banu Ormeci</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Claire-Oswald/ class=align-middle data-uni=wat>Claire Oswald</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Sarah-Jane-Payne/ class=align-middle data-uni=wat>Sarah Jane Payne</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/H/Hui-Peng/ class=align-middle data-uni=wat>Hui Peng</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Shelley-Peterson/ class=align-middle data-uni=wat>Shelley Peterson</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Art-FY-Poon/ class=align-middle data-uni=wat>Art F.Y. Poon</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/D/Denina-Simmons/ class=align-middle data-uni=wat>Denina Simmons</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jianxian-Sun/ class=align-middle data-uni=wat>Jianxian Sun</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Minqing-Yang/ class=align-middle data-uni=wat>Minqing Yang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/G/Gustavo-Ybazeta/ class=align-middle data-uni=wat>Gustavo Ybazeta</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Nathanael-BJ-Harper/ class=align-middle data-uni=wat>Nathanael B.J. Harper</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/H/Hadi-A-Dhiyebi/ class=align-middle data-uni=wat>Hadi A. Dhiyebi</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Samina-Hayat/ class=align-middle data-uni=wat>Samina Hayat</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Shelley-W-Peterson/ class=align-middle data-uni=wat>Shelley W. Peterson</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/I/Ivy-Yang/ class=align-middle data-uni=wat>Ivy Yang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/JX-Sun/ class=align-middle data-uni=wat>J.X. Sun</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li></div></ul></div></div><div class="col-12 col-md-6 col-lg-12"><div class="card my-2 my-md-0 my-lg-2"><h5 class=card-header>Venues</h5><ul class="list-group list-group-flush list-group-compact"><li class=list-group-item><a href=/gwf-publications/venues/gwf/ class=align-middle>GWF</a><span class="badge badge-secondary align-middle ml-2">5</span></li></ul></div></div></div></div></div></section></div><footer class="bg-gradient-light py-2 py-xl-3 mt-3 mt-md-4 mt-xl-5"><div class=container><p class="text-muted small px-1">Global Water Futures Publications!</p></div></footer><script src=https://code.jquery.com/jquery-3.3.1.slim.min.js integrity=sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js integrity=sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut crossorigin=anonymous></script><script src=https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js integrity=sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k crossorigin=anonymous></script><script>$(function(){$('[data-toggle="tooltip"]').tooltip();if($("#toggle-all-abstracts")){$("#toggle-all-abstracts").click(function(){var target=$("#toggle-all-abstracts");target.attr("disabled",true);if(target.attr("data-toggle-state")=="hide"){$(".abstract-collapse").collapse('show');target.attr("data-toggle-state","show");}else{$(".abstract-collapse").collapse('hide');target.attr("data-toggle-state","hide");}
target.attr("disabled",false);});$("#toggle-all-abstracts").attr("disabled",false);}})</script></body></html>