<!doctype html><html lang=en-us><head><link rel=preconnect href=https://fonts.googleapis.com><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@700&display=swap" rel=stylesheet><meta charset=utf-8><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><script src=https://ajax.googleapis.com/ajax/libs/jquery/3.6.1/jquery.min.js></script><script>window.onload=function(){document.getElementById("expand-proj").addEventListener("click",function(){this.textContent=="show less..."?this.textContent="show all...":this.textContent="show less..."}),document.getElementById("expand-uni").addEventListener("click",function(){this.textContent=="show less..."?this.textContent="show all...":this.textContent="show less..."})}</script><script>var unis={};$(document).on("change","input[type='checkbox']",function(){if(this.id.startsWith("check_year")){var e,n,t=$(':input[id^="check_year"]').map(function(){return $(this).prop("checked")}).get().includes(!0);const s=document.getElementById("main").children;if($("#year_list li:first-child").css("display")=="none"&&($("#year_list li:first-child").prop("style","display:initial !important"),$('div[id^="year_"]').prop("style","display:none !important")),t)for(child of s)child.id.startsWith(this.id.slice(-9))&&(this.checked?child.style.display="initial":child.style.display="none");else{$("#year_list li:first-child").prop("style","display:none !important");for(child of s)child.style.display="initial"}}else if(this.id.startsWith("uni"))if(console.log("Hi2"),t=$(':input[id^="uni"]').map(function(){return $(this).prop("checked")}).get().includes(!0),e=this.id.slice(-7),unis[e]==void 0&&(unis[e]=!0),this.checked?unis[e]=!0:unis[e]=!1,$("#university_list li:first-child").css("display")=="none"&&($("#university_list li:first-child").prop("style","display:initial !important"),$('div[data-prj^="prj"]').each(function(){$(this).children().first().prop("style","display:none !important")})),t){const t=document.querySelectorAll("[data-uni="+e+"]");for(child of t)if(this.checked)child.parentNode.parentNode.setAttribute("style","display:flex !important");else{disp_off_flag=!0;for(child of child.parentNode.querySelectorAll("[data-uni]"))if(unis[child.getAttribute("data-uni")]==void 0&&(unis[child.getAttribute("data-uni")]=!0),unis[child.getAttribute("data-uni")]==!0){disp_off_flag=!1;break}disp_off_flag&&child.parentNode.parentNode.setAttribute("style","display:none !important")}}else $("#university_list li:first-child").prop("style","display:none !important"),$('div[data-prj^="prj"]').each(function(){$(this).children().first().prop("style","display:flex !important")});else if(this.id.startsWith("prj"))if(t=$(':input[id^="prj"]').map(function(){return $(this).prop("checked")}).get().includes(!0),n=this.id.slice(4),$("#project_list li:first-child").css("display")=="none"&&($("#project_list li:first-child").prop("style","display:initial !important"),$('div[data-prj^="prj"]').each(function(){$(this).prop("style","display:none !important")})),t){const e=document.querySelectorAll("[data-prj="+n+"]");for(child of e)this.checked?child.setAttribute("style","display:initial !important"):child.setAttribute("style","display:none !important")}else console.log("all unckecked"),$("#project_list li:first-child").prop("style","display:none !important"),$('div[data-prj^="prj"]').each(function(){$(this).prop("style","display:initial !important")})});function clearAll(e){e=="year"?($(':input[id^="check_year"]').prop("checked",!1),$('div[id^="year_"]').prop("style","display:initial;"),$("#year_list li:first-child").prop("style","display:none !important")):e=="uni"?($(':input[id^="uni"]').prop("checked",!1),$('div[data-prj^="prj"]').each(function(){$(this).children().first().prop("style","display:flex !important")}),$("#university_list li:first-child").prop("style","display:none !important")):e=="prj"&&($(':input[id^="prj"]').prop("checked",!1),$('div[data-prj^="prj"]').each(function(){$(this).prop("style","display:initial !important")}),$("#project_list li:first-child").prop("style","display:none !important"))}</script><style>.arrowup{border:solid #fff;border-width:0 3px 3px 0;display:inline-block;padding:5px;transform:rotate(-135deg);-webkit-transform:rotate(-135deg)}#topBtn{display:none;position:fixed;bottom:20px;right:30px;z-index:99;font-size:30px;border:none;outline:none;background-color:#29abe0;color:#fff;cursor:pointer;padding:15px;border-radius:8px;height:10dp}#topBtn:hover{background-color:#1b8dbb}.card-header-custom{cursor:pointer;-webkit-user-select:none;-ms-user-select:none;user-select:none}.clear-widget{color:#747474!important;background-color:#dee2e6;font-size:80%!important}.clear-widget:hover{color:#212529!important;background-color:#c1c9d0;cursor:pointer}</style><!--[if IEMobile]><meta http-equiv=cleartype content="on"><![endif]--><title>Chand S. Mangat - GWF Publications</title>
<meta name=generator content="Hugo 0.144.1"><link href=/gwf-publications/gwficon.ico rel="shortcut icon" type=image/x-icon><link rel=stylesheet href=/gwf-publications/css/main.min.b1d14a9a8f6bb9c608ca4de9aad72a6e06945119f97951f2908522dc220e6277.css media=screen><link rel=stylesheet href=https://use.fontawesome.com/releases/v5.7.2/css/all.css integrity=sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr crossorigin=anonymous><link rel=stylesheet href=/css/academicons.min.css></head><body><header style='display:block;box-sizing:border-box;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;line-height:1.5;color:#222;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%'><div id=navbar-container style='height:54px;width:100%;background:rgba(255,255,255,.95);overflow:hidden;-webkit-box-shadow:rgba(0,0,0,9%)0 1px 0;box-shadow:rgba(0,0,0,9%)0 1px;box-sizing:border-box;display:block;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;line-height:1.5;color:#222;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%'><a href=/gwf-publications style='color:#0072bc;text-decoration:none;display:inline-block;height:100%;line-height:54px;border-right:1px solid #e6e6e6;margin-left:6px;padding-right:6px;outline-offset:-4px!important;font-weight:700;background:0 0;box-sizing:border-box;cursor:pointer;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;-webkit-tap-highlight-color:transparent'><img src=/gwf-publications/images/gwf-logo.png alt="GWF Logo" style='margin-right:0;height:42px;padding:4px;width:auto;margin-top:-2px;max-width:100%;vertical-align:middle;border:0;box-sizing:border-box;color:#0072bc;line-height:54px;font-weight:700;cursor:pointer;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%'>
</a><a href=/gwf-publications style='text-decoration:none;color:#222!important;display:inline-block;font-size:16px;line-height:54px;font-weight:700;outline-offset:-4px!important;padding:0 6px;margin:0 -6px;background:0 0;box-sizing:border-box;cursor:pointer;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%;padding-left:12px'>Global Water Futures</a></div></header><div id=main-container class=container><section id=main><h2 id=title><span class=font-weight-normal>Chand S.</span> <span class=font-weight-bold>Mangat</span></h2><hr><div class=row><div class=col-lg-9><h4>2023</h4><div data-prj=prj1><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1016/j.scitotenv.2023.163292 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G23-27001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G23-27001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G23-27001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G23-27001/>An improved method for determining frequency of multiple variants of SARS-CoV-2 in wastewater using qPCR assays</a></strong><br><a href=/gwf-publications/people/M/Meghan-Fuzzen/ data-uni=uni0002>Meghan Fuzzen</a>,
<a href=/gwf-publications/people/N/Nathanael-BJ-Harper/ data-uni=uni0002>Nathanael B.J. Harper</a>,
<a href=/gwf-publications/people/H/Hadi-A-Dhiyebi/ data-uni=uni0002>Hadi A. Dhiyebi</a>,
<a href=/gwf-publications/people/N/Nivetha-Srikanthan/ data-uni=uni0155>Nivetha Srikanthan</a>,
<a href=/gwf-publications/people/S/Samina-Hayat/ data-uni=uni0002>Samina Hayat</a>,
<a href=/gwf-publications/people/L/Leslie-M-Bragg/ data-uni=uni0002>Leslie M. Bragg</a>,
<a href=/gwf-publications/people/S/Shelley-Peterson/ data-uni=uni0104>Shelley Peterson</a>,
<a href=/gwf-publications/people/M/Minqing-Ivy-Yang/ data-uni=uni0031>Minqing Ivy Yang</a>,
<a href=/gwf-publications/people/J/Jianxian-Sun/ data-uni=uni0031>Jianxian Sun</a>,
<a href=/gwf-publications/people/E/Elizabeth-A-Edwards/ data-uni=uni0031>Elizabeth A. Edwards</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0001>John P. Giesy</a>,
<a href=/gwf-publications/people/C/Chand-S-Mangat/ data-uni=uni0104>Chand S. Mangat</a>,
<a href=/gwf-publications/people/T/Tyson-E-Graber/ data-uni=uni0114>Tyson E. Graber</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=uni0028>Robert Delatolla</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/M/Meghan-Fuzzen/ data-uni=uni0002>Meghan Fuzzen</a>,
<a href=/gwf-publications/people/N/Nathanael-BJ-Harper/ data-uni=uni0002>Nathanael B.J. Harper</a>,
<a href=/gwf-publications/people/H/Hadi-A-Dhiyebi/ data-uni=uni0002>Hadi A. Dhiyebi</a>,
<a href=/gwf-publications/people/N/Nivetha-Srikanthan/ data-uni=uni0155>Nivetha Srikanthan</a>,
<a href=/gwf-publications/people/S/Samina-Hayat/ data-uni=uni0002>Samina Hayat</a>,
<a href=/gwf-publications/people/L/Leslie-M-Bragg/ data-uni=uni0002>Leslie M. Bragg</a>,
<a href=/gwf-publications/people/S/Shelley-Peterson/ data-uni=uni0104>Shelley Peterson</a>,
<a href=/gwf-publications/people/M/Minqing-Ivy-Yang/ data-uni=uni0031>Minqing Ivy Yang</a>,
<a href=/gwf-publications/people/J/Jianxian-Sun/ data-uni=uni0031>Jianxian Sun</a>,
<a href=/gwf-publications/people/E/Elizabeth-A-Edwards/ data-uni=uni0031>Elizabeth A. Edwards</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0001>John P. Giesy</a>,
<a href=/gwf-publications/people/C/Chand-S-Mangat/ data-uni=uni0104>Chand S. Mangat</a>,
<a href=/gwf-publications/people/T/Tyson-E-Graber/ data-uni=uni0114>Tyson E. Graber</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=uni0028>Robert Delatolla</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a><br><a href=/gwf-publications/volumes/G23-27/ class=text-muted>Science of The Total Environment, Volume 881</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G23-27001><div class="card-body p-3 small">Wastewater-based surveillance has become an effective tool around the globe for indirect monitoring of COVID-19 in communities. Variants of Concern (VOCs) have been detected in wastewater by use of reverse transcription polymerase chain reaction (RT-PCR) or whole genome sequencing (WGS). Rapid, reliable RT-PCR assays continue to be needed to determine the relative frequencies of VOCs and sub-lineages in wastewater-based surveillance programs. The presence of multiple mutations in a single region of the N-gene allowed for the design of a single amplicon, multiple probe assay, that can distinguish among several VOCs in wastewater RNA extracts. This approach which multiplexes probes designed to target mutations associated with specific VOC's along with an intra-amplicon universal probe (non-mutated region) was validated in singleplex and multiplex. The prevalence of each mutation (i.e. VOC) is estimated by comparing the abundance of the targeted mutation with a non-mutated and highly conserved region within the same amplicon. This is advantageous for the accurate and rapid estimation of variant frequencies in wastewater. The N200 assay was applied to monitor frequencies of VOCs in wastewater extracts from several communities in Ontario, Canada in near real time from November 28, 2021 to January 4, 2022. This includes the period of the rapid replacement of the Delta variant with the introduction of the Omicron variant in these Ontario communities in early December 2021. The frequency estimates using this assay were highly reflective of clinical WGS estimates for the same communities. This style of qPCR assay, which simultaneously measures signal from a non-mutated comparator probe and multiple mutation-specific probes contained within a single qPCR amplicon, can be applied to future assay development for rapid and accurate estimations of variant frequencies.</div></div><div data-prj=prj23><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.3389/fmicb.2023.1048661 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G23-68001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G23-68001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G23-68001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G23-68001/>Realizing the value in “non-standard” parts of the qPCR standard curve by integrating fundamentals of quantitative microbiology</a></strong><br><a href=/gwf-publications/people/P/Philip-J-Schmidt/ data-uni=uni0002>Philip J. Schmidt</a>,
<a href=/gwf-publications/people/N/Nicole-Acosta/ data-uni=uni0007>Nicole Acosta</a>,
<a href=/gwf-publications/people/A/Alex-H-S-Chik/ data-uni=uni0433>Alex H. S. Chik</a>,
<a href=/gwf-publications/people/P/Patrick-M-DAoust/ data-uni=uni0028>Patrick M. D’Aoust</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=uni0028>Robert Delatolla</a>,
<a href=/gwf-publications/people/H/Hadi-A-Dhiyebi/ data-uni=uni0002>Hadi A. Dhiyebi</a>,
<a href=/gwf-publications/people/M/Melissa-B-Glier/ data-uni=uni0518>Melissa B. Glier</a>,
<a href=/gwf-publications/people/C/Casey-R-J-Hubert/ data-uni=uni0007>Casey R. J. Hubert</a>,
<a href=/gwf-publications/people/J/Jennifer-Kopetzky/ data-uni=uni1060>Jennifer Kopetzky</a>,
<a href=/gwf-publications/people/C/Chand-S-Mangat/ data-uni=uni0104>Chand S. Mangat</a>,
<a href=/gwf-publications/people/X/Xiaoli-Pang/ data-uni=uni0015>Xiaoli Pang</a>,
<a href=/gwf-publications/people/S/Shelley-Peterson/ data-uni=uni0104>Shelley Peterson</a>,
<a href=/gwf-publications/people/N/Natalie-Prystajecky/ data-uni=uni0013>Natalie Prystajecky</a>,
<a href=/gwf-publications/people/Y/Yuanyuan-Qiu/ data-uni=uni0432>Yuanyuan Qiu</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/M/Monica-B-Emelko/ data-uni=uni0002>Monica B. Emelko</a>,
<a href=/gwf-publications/people/P/Philip-J-Schmidt/ data-uni=uni0002>Philip J. Schmidt</a>,
<a href=/gwf-publications/people/N/Nicole-Acosta/ data-uni=uni0007>Nicole Acosta</a>,
<a href=/gwf-publications/people/A/Alex-H-S-Chik/ data-uni=uni0433>Alex H. S. Chik</a>,
<a href=/gwf-publications/people/P/Patrick-M-DAoust/ data-uni=uni0028>Patrick M. D’Aoust</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=uni0028>Robert Delatolla</a>,
<a href=/gwf-publications/people/H/Hadi-A-Dhiyebi/ data-uni=uni0002>Hadi A. Dhiyebi</a>,
<a href=/gwf-publications/people/M/Melissa-B-Glier/ data-uni=uni0518>Melissa B. Glier</a>,
<a href=/gwf-publications/people/C/Casey-R-J-Hubert/ data-uni=uni0007>Casey R. J. Hubert</a>,
<a href=/gwf-publications/people/J/Jennifer-Kopetzky/ data-uni=uni1060>Jennifer Kopetzky</a>,
<a href=/gwf-publications/people/C/Chand-S-Mangat/ data-uni=uni0104>Chand S. Mangat</a>,
<a href=/gwf-publications/people/X/Xiaoli-Pang/ data-uni=uni0015>Xiaoli Pang</a>,
<a href=/gwf-publications/people/S/Shelley-Peterson/ data-uni=uni0104>Shelley Peterson</a>,
<a href=/gwf-publications/people/N/Natalie-Prystajecky/ data-uni=uni0013>Natalie Prystajecky</a>,
<a href=/gwf-publications/people/Y/Yuanyuan-Qiu/ data-uni=uni0432>Yuanyuan Qiu</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/M/Monica-B-Emelko/ data-uni=uni0002>Monica B. Emelko</a><br><a href=/gwf-publications/volumes/G23-68/ class=text-muted>Frontiers in Microbiology, Volume 14</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G23-68001><div class="card-body p-3 small">The real-time polymerase chain reaction (PCR), commonly known as quantitative PCR (qPCR), is increasingly common in environmental microbiology applications. During the COVID-19 pandemic, qPCR combined with reverse transcription (RT-qPCR) has been used to detect and quantify SARS-CoV-2 in clinical diagnoses and wastewater monitoring of local trends. Estimation of concentrations using qPCR often features a log-linear standard curve model calibrating quantification cycle (Cq) values obtained from underlying fluorescence measurements to standard concentrations. This process works well at high concentrations within a linear dynamic range but has diminishing reliability at low concentrations because it cannot explain "non-standard" data such as Cq values reflecting increasing variability at low concentrations or non-detects that do not yield Cq values at all. Here, fundamental probabilistic modeling concepts from classical quantitative microbiology were integrated into standard curve modeling approaches by reflecting well-understood mechanisms for random error in microbial data. This work showed that data diverging from the log-linear regression model at low concentrations as well as non-detects can be seamlessly integrated into enhanced standard curve analysis. The newly developed model provides improved representation of standard curve data at low concentrations while converging asymptotically upon conventional log-linear regression at high concentrations and adding no fitting parameters. Such modeling facilitates exploration of the effects of various random error mechanisms in experiments generating standard curve data, enables quantification of uncertainty in standard curve parameters, and is an important step toward quantifying uncertainty in qPCR-based concentration estimates. Improving understanding of the random error in qPCR data and standard curve modeling is especially important when low concentrations are of particular interest and inappropriate analysis can unduly affect interpretation, conclusions regarding lab performance, reported concentration estimates, and associated decision-making.</div></div><h4>2022</h4><div data-prj=prj1><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.21203/rs.3.rs-1439969/v2 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G22-18001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G22-18001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G22-18001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G22-18001/>Community Surveillance of Omicron in Ontario: Wastewater-based Epidemiology Comes of Age.</a></strong><br><a href=/gwf-publications/people/A/Authors-presented-in-alphabetical-order/ data-uni=uni0000>Authors presented in alphabetical order</a>,
<a href=/gwf-publications/people/E/Eric-J-Arts/ data-uni=uni0035>Eric J. Arts</a>,
<a href=/gwf-publications/people/R/R-Stephen-Brown/ data-uni=uni0049>R. Stephen Brown</a>,
<a href=/gwf-publications/people/D/David-Bulir/ data-uni=uni0004>David Bulir</a>,
<a href=/gwf-publications/people/T/Trevor-C-Charles/ data-uni=uni1437>Trevor C. Charles</a>,
<a href=/gwf-publications/people/C/Christopher-T-DeGroot/ data-uni=uni0035>Christopher T. DeGroot</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=uni0028>Robert Delatolla</a>,
<a href=/gwf-publications/people/J/JeanPaul-Desaulniers/ data-uni=uni0448>Jean‐Paul Desaulniers</a>,
<a href=/gwf-publications/people/E/Elizabeth-A-Edwards/ data-uni=uni0031>Elizabeth A. Edwards</a>,
<a href=/gwf-publications/people/M/Meghan-Fuzzen/ data-uni=uni0002>Meghan Fuzzen</a>,
<a href=/gwf-publications/people/K/Kimberley-Gilbride/ data-uni=uni0163>Kimberley Gilbride</a>,
<a href=/gwf-publications/people/J/Jodi-Gilchrist/ data-uni=uni1438>Jodi Gilchrist</a>,
<a href=/gwf-publications/people/L/Lawrence-Goodridge/ data-uni=uni0011>Lawrence Goodridge</a>,
<a href=/gwf-publications/people/T/Tyson-E-Graber/ data-uni=uni0114>Tyson E. Graber</a>,
<a href=/gwf-publications/people/M/Marc-Habash/ data-uni=uni1007>Marc Habash</a>,
<a href=/gwf-publications/people/P/Peter-J%C3%BCni/ data-uni=uni0085>Peter Jüni</a>,
<a href=/gwf-publications/people/A/Andrea-E-Kirkwood/ data-uni=uni0448>Andrea E. Kirkwood</a>,
<a href=/gwf-publications/people/J/James-Knockleby/ data-uni=uni0449>James Knockleby</a>,
<a href=/gwf-publications/people/C/Christopher-J-Kyle/ data-uni=uni0074>Christopher J. Kyle</a>,
<a href=/gwf-publications/people/C/Chrystal-Landgraff/ data-uni=uni0104>Chrystal Landgraff</a>,
<a href=/gwf-publications/people/C/Chand-S-Mangat/ data-uni=uni0104>Chand S. Mangat</a>,
<a href=/gwf-publications/people/D/Douglas-G-Manuel/ data-uni=uni0028>Douglas G. Manuel</a>,
<a href=/gwf-publications/people/R/R-Michael-L-McKay/ data-uni=uni0169>R. Michael L. McKay</a>,
<a href=/gwf-publications/people/E/Edgard-M-Mejia/ data-uni=uni0014>Edgard M. Mejia</a>,
<a href=/gwf-publications/people/A/Aleksandra-Mloszewska/ data-uni=uni0449>Aleksandra Mloszewska</a>,
<a href=/gwf-publications/people/B/Banu-%C3%96rmeci/ data-uni=uni0069>Banu Örmeci</a>,
<a href=/gwf-publications/people/C/Claire-Oswald/ data-uni=uni0163>Claire Oswald</a>,
<a href=/gwf-publications/people/S/Sarah-Jane-Payne/ data-uni=uni0049>Sarah Jane Payne</a>,
<a href=/gwf-publications/people/H/Hui-Peng/ data-uni=uni0031>Hui Peng</a>,
<a href=/gwf-publications/people/S/Shelley-Peterson/ data-uni=uni0104>Shelley Peterson</a>,
<a href=/gwf-publications/people/A/Art-F-Y-Poon/ data-uni=uni0035>Art F. Y. Poon</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/D/Denina-Simmons/ data-uni=uni0448>Denina Simmons</a>,
<a href=/gwf-publications/people/J/Jianxian-Sun/ data-uni=uni0031>Jianxian Sun</a>,
<a href=/gwf-publications/people/M/Minqing-Ivy-Yang/ data-uni=uni0031>Minqing Ivy Yang</a>,
<a href=/gwf-publications/people/G/Gustavo-Ybazeta/ data-uni=uni0449>Gustavo Ybazeta</a><br><a href=/gwf-publications/volumes/G22-18/ class=text-muted></a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G22-18001><div class="card-body p-3 small">Abstract Wastewater-based surveillance of SARS-CoV-2 RNA has been implemented at building, neighbourhood, and city levels throughout the world. Implementation strategies and analysis methods differ, but they all aim to provide rapid and reliable information about community COVID-19 health states. A viable and sustainable SARS-CoV-2 surveillance network must not only provide reliable and timely information about COVID-19 trends, but also provide for scalability as well as accurate detection of known or unknown emerging variants. Emergence of the SARS-CoV-2 variant of concern Omicron in late Fall 2021 presented an excellent opportunity to benchmark individual and aggregated data outputs of the Ontario Wastewater Surveillance Initiative in Canada; this public health-integrated surveillance network monitors wastewaters from over 10 million people across major population centres of the province. We demonstrate that this coordinated approach provides excellent situational awareness, comparing favourably with traditional clinical surveillance measures. Thus, aggregated datasets compiled from multiple wastewater-based surveillance nodes can provide sufficient sensitivity (i.e., early indication of increasing and decreasing incidence of SARS-CoV-2) and specificity (i.e., allele frequency estimation of emerging variants) with which to make informed public health decisions at regional- and state-levels.</div></div><div data-prj=prj1><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1101/2022.04.12.22273761 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G22-18002.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G22-18002 data-toggle=collapse aria-expanded=false aria-controls=abstract-G22-18002 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G22-18002/>Multiplex RT-qPCR assay (N200) to detect and estimate prevalence of multiple SARS-CoV-2 Variants of Concern in wastewater</a></strong><br><a href=/gwf-publications/people/M/Meghan-Fuzzen/ data-uni=uni0002>Meghan Fuzzen</a>,
<a href=/gwf-publications/people/N/Nathanael-BJ-Harper/ data-uni=uni0002>Nathanael B.J. Harper</a>,
<a href=/gwf-publications/people/H/Hadi-A-Dhiyebi/ data-uni=uni0002>Hadi A. Dhiyebi</a>,
<a href=/gwf-publications/people/N/Nivetha-Srikanthan/ data-uni=uni0155>Nivetha Srikanthan</a>,
<a href=/gwf-publications/people/S/Samina-Hayat/ data-uni=uni0002>Samina Hayat</a>,
<a href=/gwf-publications/people/S/Shelley-Peterson/ data-uni=uni0104>Shelley Peterson</a>,
<a href=/gwf-publications/people/M/Minqing-Ivy-Yang/ data-uni=uni0031>Minqing Ivy Yang</a>,
<a href=/gwf-publications/people/J/Jianxian-Sun/ data-uni=uni0031>Jianxian Sun</a>,
<a href=/gwf-publications/people/E/Elizabeth-A-Edwards/ data-uni=uni0031>Elizabeth A. Edwards</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0001>John P. Giesy</a>,
<a href=/gwf-publications/people/C/Chand-S-Mangat/ data-uni=uni0104>Chand S. Mangat</a>,
<a href=/gwf-publications/people/T/Tyson-E-Graber/ data-uni=uni0114>Tyson E. Graber</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=uni0028>Robert Delatolla</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a><br><a href=/gwf-publications/volumes/G22-18/ class=text-muted></a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G22-18002><div class="card-body p-3 small">Abstract Wastewater-based surveillance (WBS) has become an effective tool around the globe for indirect monitoring of COVID-19 in communities. Quantities of viral fragments of SARS-CoV-2 in wastewater are related to numbers of clinical cases of COVID-19 reported within the corresponding sewershed. Variants of Concern (VOCs) have been detected in wastewater by use of reverse transcription quantitative polymerase chain reaction (RT-qPCR) or sequencing. A multiplex RT-qPCR assay to detect and estimate the prevalence of multiple VOCs, including Omicron/Alpha, Beta, Gamma, and Delta, in wastewater RNA extracts was developed and validated. The probe-based multiplex assay, named “N200” focuses on amino acids 199-202, a region of the N gene that contains several mutations that are associated with variants of SARS- CoV-2 within a single amplicon. Each of the probes in the N200 assay are specific to the targeted mutations and worked equally well in single- and multi-plex modes. To estimate prevalence of each VOC, the abundance of the targeted mutation was compared with a non- mutated region within the same amplified region. The N200 assay was applied to monitor frequencies of VOCs in wastewater extracts from six sewersheds in Ontario, Canada collected between December 1, 2021, and January 4, 2022. Using the N200 assay, the replacement of the Delta variant along with the introduction and rapid dominance of the Omicron variant were monitored in near real-time, as they occurred nearly simultaneously at all six locations. The N200 assay is robust and efficient for wastewater surveillance can be adopted into VOC monitoring programs or replace more laborious assays currently being used to monitor SARS- CoV-2 and its VOCs.</div></div><div data-prj=prj1><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1021/acsestwater.1c00349 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G22-21001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G22-21001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G22-21001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G22-21001/>RNA in Municipal Wastewater Reveals Magnitudes of COVID-19 Outbreaks across Four Waves Driven by SARS-CoV-2 Variants of Concern</a></strong><br><a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0056>Yuwei Xie</a>,
<a href=/gwf-publications/people/J/Jonathan-K-Challis/ data-uni=uni0034>Jonathan K. Challis</a>,
<a href=/gwf-publications/people/F/Femi-F-Oloye/ data-uni=uni0226>Femi F. Oloye</a>,
<a href=/gwf-publications/people/M/Mohsen-Asadi/ data-uni=uni0001>Mohsen Asadi</a>,
<a href=/gwf-publications/people/J/Jenna-Cantin/ data-uni=uni0001>Jenna Cantin</a>,
<a href=/gwf-publications/people/M/Markus-Brinkmann/ data-uni=uni0001>Markus Brinkmann</a>,
<a href=/gwf-publications/people/K/Kerry-N-McPhedran/ data-uni=uni0001>Kerry N. McPhedran</a>,
<a href=/gwf-publications/people/N/Natacha-S-Hogan/ data-uni=uni0001>Natacha S. Hogan</a>,
<a href=/gwf-publications/people/M/Mike-Sadowski/ data-uni=uni1027>Mike Sadowski</a>,
<a href=/gwf-publications/people/P/Paul-D-Jones/ data-uni=uni0001>Paul D. Jones</a>,
<a href=/gwf-publications/people/C/Chrystal-Landgraff/ data-uni=uni0104>Chrystal Landgraff</a>,
<a href=/gwf-publications/people/C/Chand-S-Mangat/ data-uni=uni0104>Chand S. Mangat</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0001>John P. Giesy</a><br><a href=/gwf-publications/volumes/G22-21/ class=text-muted>ACS ES&amp;T Water, Volume 2, Issue 11</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G22-21001><div class="card-body p-3 small">There are no standardized protocols for quantifying severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in wastewater to date, especially for population normalization. Here, a pipeline was developed, applied, and assessed to quantify SARS-CoV-2 and key variants of concern (VOCs) RNA in wastewater at Saskatoon, Canada. Normalization approaches using recovery ratio and extraction efficiency, wastewater parameters, or population indicators were assessed by comparing to daily numbers of new cases. Viral load was positively correlated with daily new cases reported in the sewershed. Wastewater surveillance (WS) had a lead time of approximately 7 days, which indicated surges in the number of new cases. WS revealed the variant α and δ driving the third and fourth wave, respectively. The adjustment with the recovery ratio and extraction efficiency improved the correlation between viral load and daily new cases. Normalization of viral concentration to concentrations of the artificial sweetener acesulfame K improved the trend of viral load during the Christmas and New Year holidays when populations were dynamic and variable. Acesulfame K performed better than pepper mild mottle virus, creatinine, and ammonia for population normalization. Hence, quality controls to characterize recovery ratios and extraction efficiencies and population normalization with acesulfame are promising for precise WS programs supporting decision-making in public health.</div></div><div data-prj=prj1><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1016/j.scitotenv.2022.156741 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G22-152002.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G22-152002 data-toggle=collapse aria-expanded=false aria-controls=abstract-G22-152002 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G22-152002/>Rapid transition between SARS-CoV-2 variants of concern Delta and Omicron detected by monitoring municipal wastewater from three Canadian cities</a></strong><br><a href=/gwf-publications/people/F/Femi-F-Oloye/ data-uni=uni0226>Femi F. Oloye</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0056>Yuwei Xie</a>,
<a href=/gwf-publications/people/M/Mohsen-Asadi/ data-uni=uni0001>Mohsen Asadi</a>,
<a href=/gwf-publications/people/J/Jenna-Cantin/ data-uni=uni0001>Jenna Cantin</a>,
<a href=/gwf-publications/people/J/Jonathan-K-Challis/ data-uni=uni0034>Jonathan K. Challis</a>,
<a href=/gwf-publications/people/M/Markus-Brinkmann/ data-uni=uni0001>Markus Brinkmann</a>,
<a href=/gwf-publications/people/K/Kerry-N-McPhedran/ data-uni=uni0001>Kerry N. McPhedran</a>,
<a href=/gwf-publications/people/K/Kevin-Kristian/ data-uni=uni1516>Kevin Kristian</a>,
<a href=/gwf-publications/people/M/Mark-P-Keller/ data-uni=uni0046>Mark P. Keller</a>,
<a href=/gwf-publications/people/M/Mike-Sadowski/ data-uni=uni1027>Mike Sadowski</a>,
<a href=/gwf-publications/people/P/Paul-D-Jones/ data-uni=uni0001>Paul D. Jones</a>,
<a href=/gwf-publications/people/C/Chrystal-Landgraff/ data-uni=uni0104>Chrystal Landgraff</a>,
<a href=/gwf-publications/people/C/Chand-S-Mangat/ data-uni=uni0104>Chand S. Mangat</a>,
<a href=/gwf-publications/people/M/Meghan-Fuzzen/ data-uni=uni0002>Meghan Fuzzen</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0001>John P. Giesy</a><br><a href=/gwf-publications/volumes/G22-152/ class=text-muted>Science of The Total Environment, Volume 841</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G22-152002><div class="card-body p-3 small">Monitoring the communal incidence of COVID-19 is important for both government and residents of an area to make informed decisions. However, continuous reliance on one means of monitoring might not be accurate because of biases introduced by government policies or behaviours of residents. Wastewater surveillance was employed to monitor concentrations of SARS-CoV-2 RNA in raw influent wastewater from wastewater treatment plants serving three Canadian Prairie cities with different population sizes. Data obtained from wastewater are not directly influenced by government regulations or behaviours of individuals. The means of three weekly samples collected using 24 h composite auto-samplers were determined. Viral loads were determined by RT-qPCR, and whole-genome sequencing was used to charaterize variants of concern (VOC). The dominant VOCs in the three cities were the same but with different proportions of sub-lineages. Sub-lineages of Delta were AY.12, AY.25, AY.27 and AY.93 in 2021, while the major sub-lineage of Omicron was BA.1 in January 2022, and BA.2 subsequently became a trace-level sub-variant then the predominant VOC. When each VOC was first detected varied among cities; However, Saskatoon, with the largest population, was always the first to present new VOCs. Viral loads varied among cities, but there was no direct correlation with population size, possibly because of differences in flow regimes. Population is one of the factors that affects trends in onset and development of local outbreaks during the pandemic. This might be due to demography or the fact that larger populations had greater potential for inter- and intra-country migration. Hence, wastewater surveillance data from larger cities can typically be used to indicate what to expect in smaller communities.</div></div><h4>2021</h4><div data-prj=prj1><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1016/j.jes.2021.01.029 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-41001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-41001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-41001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-41001/>Comparison of approaches to quantify SARS-CoV-2 in wastewater using RT-qPCR: Results and implications from a collaborative inter-laboratory study in Canada</a></strong><br><a href=/gwf-publications/people/A/Alex-H-S-Chik/ data-uni=uni0433>Alex H. S. Chik</a>,
<a href=/gwf-publications/people/M/Melissa-B-Glier/ data-uni=uni0518>Melissa B. Glier</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/C/Chand-S-Mangat/ data-uni=uni0104>Chand S. Mangat</a>,
<a href=/gwf-publications/people/X/Xiaoli-Pang/ data-uni=uni0015>Xiaoli Pang</a>,
<a href=/gwf-publications/people/Y/Yuanyuan-Qiu/ data-uni=uni0432>Yuanyuan Qiu</a>,
<a href=/gwf-publications/people/P/Patrick-M-DAoust/ data-uni=uni0028>Patrick M. D’Aoust</a>,
<a href=/gwf-publications/people/J/JeanBaptiste-Burnet/ data-uni=uni0190>Jean‐Baptiste Burnet</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=uni0028>Robert Delatolla</a>,
<a href=/gwf-publications/people/S/Sarah-Dorner/ data-uni=uni0190>Sarah Dorner</a>,
<a href=/gwf-publications/people/Q/Qiudi-Geng/ data-uni=uni0169>Qiudi Geng</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0001>John P. Giesy</a>,
<a href=/gwf-publications/people/R/R-Michael-L-McKay/ data-uni=uni0169>R. Michael L. McKay</a>,
<a href=/gwf-publications/people/M/Michael-R-Mulvey/ data-uni=uni0014>Michael R. Mulvey</a>,
<a href=/gwf-publications/people/N/Natalie-Prystajecky/ data-uni=uni0013>Natalie Prystajecky</a>,
<a href=/gwf-publications/people/N/Nivetha-Srikanthan/ data-uni=uni0155>Nivetha Srikanthan</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0056>Yuwei Xie</a>,
<a href=/gwf-publications/people/B/Bernadette-Conant/ data-uni=uni0935>Bernadette Conant</a>,
<a href=/gwf-publications/people/S/Steve-E-Hrudey/ data-uni=uni0015>Steve E. Hrudey</a>,
<a href=/gwf-publications/people/A/Alex-H-S-Chik/ data-uni=uni0433>Alex H. S. Chik</a>,
<a href=/gwf-publications/people/M/Melissa-B-Glier/ data-uni=uni0518>Melissa B. Glier</a>,
<a href=/gwf-publications/people/M/Mark-R-Servos/ data-uni=uni0002>Mark R. Servos</a>,
<a href=/gwf-publications/people/C/Chand-S-Mangat/ data-uni=uni0104>Chand S. Mangat</a>,
<a href=/gwf-publications/people/X/Xiaoli-Pang/ data-uni=uni0015>Xiaoli Pang</a>,
<a href=/gwf-publications/people/Y/Yuanyuan-Qiu/ data-uni=uni0432>Yuanyuan Qiu</a>,
<a href=/gwf-publications/people/P/Patrick-M-DAoust/ data-uni=uni0028>Patrick M. D’Aoust</a>,
<a href=/gwf-publications/people/J/JeanBaptiste-Burnet/ data-uni=uni0190>Jean‐Baptiste Burnet</a>,
<a href=/gwf-publications/people/R/Robert-Delatolla/ data-uni=uni0028>Robert Delatolla</a>,
<a href=/gwf-publications/people/S/Sarah-Dorner/ data-uni=uni0190>Sarah Dorner</a>,
<a href=/gwf-publications/people/Q/Qiudi-Geng/ data-uni=uni0169>Qiudi Geng</a>,
<a href=/gwf-publications/people/J/John-P-Giesy/ data-uni=uni0001>John P. Giesy</a>,
<a href=/gwf-publications/people/R/R-Michael-L-McKay/ data-uni=uni0169>R. Michael L. McKay</a>,
<a href=/gwf-publications/people/M/Michael-R-Mulvey/ data-uni=uni0014>Michael R. Mulvey</a>,
<a href=/gwf-publications/people/N/Natalie-Prystajecky/ data-uni=uni0013>Natalie Prystajecky</a>,
<a href=/gwf-publications/people/N/Nivetha-Srikanthan/ data-uni=uni0155>Nivetha Srikanthan</a>,
<a href=/gwf-publications/people/Y/Yuwei-Xie/ data-uni=uni0056>Yuwei Xie</a>,
<a href=/gwf-publications/people/B/Bernadette-Conant/ data-uni=uni0935>Bernadette Conant</a>,
<a href=/gwf-publications/people/S/Steve-E-Hrudey/ data-uni=uni0015>Steve E. Hrudey</a><br><a href=/gwf-publications/volumes/G21-41/ class=text-muted>Journal of Environmental Sciences, Volume 107</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-41001><div class="card-body p-3 small">Detection of SARS-CoV-2 RNA in wastewater is a promising tool for informing public health decisions during the COVID-19 pandemic. However, approaches for its analysis by use of reverse transcription quantitative polymerase chain reaction (RT-qPCR) are still far from standardized globally. To characterize inter- and intra-laboratory variability among results when using various methods deployed across Canada, aliquots from a real wastewater sample were spiked with surrogates of SARS-CoV-2 (gamma-radiation inactivated SARS-CoV-2 and human coronavirus strain 229E [HCoV-229E]) at low and high levels then provided "blind" to eight laboratories. Concentration estimates reported by individual laboratories were consistently within a 1.0-log10 range for aliquots of the same spiked condition. All laboratories distinguished between low- and high-spikes for both surrogates. As expected, greater variability was observed in the results amongst laboratories than within individual laboratories, but SARS-CoV-2 RNA concentration estimates for each spiked condition remained mostly within 1.0-log10 ranges. The no-spike wastewater aliquots provided yielded non-detects or trace levels (&lt;20 gene copies/mL) of SARS-CoV-2 RNA. Detections appear linked to methods that included or focused on the solids fraction of the wastewater matrix and might represent in-situ SARS-CoV-2 to the wastewater sample. HCoV-229E RNA was not detected in the no-spike aliquots. Overall, all methods yielded comparable results at the conditions tested. Partitioning behavior of SARS-CoV-2 and spiked surrogates in wastewater should be considered to evaluate method effectiveness. A consistent method and laboratory to explore wastewater SARS-CoV-2 temporal trends for a given system, with appropriate quality control protocols and documented in adequate detail should succeed.</div></div></div><div class=col-lg-3><a class="btn btn-lg btn-secondary btn-block mb-2" href="https://www.semanticscholar.org/search?q=Chand+S.+Mangat" title="Search for 'Chand S. Mangat' on Semantic Scholar"><i class="ai ai-semantic-scholar"></i><span class=pl-sm-2>Search</span></a><div class=row><div class="col-12 col-md-6 col-lg-12"><div class=card><h5 class=card-header>Co-authors</h5><ul class="list-group list-group-flush list-group-compact"><li class=list-group-item><a href=/gwf-publications/people/M/Mark-R-Servos/ class=align-middle data-uni=uni0002>Mark R. Servos</a>
<span class="badge badge-secondary align-middle ml-2">16</span></li><li class=list-group-item><a href=/gwf-publications/people/R/Robert-Delatolla/ class=align-middle data-uni=uni0028>Robert Delatolla</a>
<span class="badge badge-secondary align-middle ml-2">14</span></li><li class=list-group-item><a href=/gwf-publications/people/J/John-P-Giesy/ class=align-middle data-uni=uni0001>John P. Giesy</a>
<span class="badge badge-secondary align-middle ml-2">11</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Shelley-Peterson/ class=align-middle data-uni=uni0104>Shelley Peterson</a>
<span class="badge badge-secondary align-middle ml-2">10</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Nivetha-Srikanthan/ class=align-middle data-uni=uni0155>Nivetha Srikanthan</a>
<span class="badge badge-secondary align-middle ml-2">9</span></li><li class="list-group-item list-group-toggle-btn py-1" data-toggle=collapse data-target=#more-coauthors aria-expanded=false aria-controls=more-coauthors>show all...</li><div class="collapse border-top" id=more-coauthors><li class=list-group-item><a href=/gwf-publications/people/H/Hadi-A-Dhiyebi/ class=align-middle data-uni=uni0002>Hadi A. Dhiyebi</a>
<span class="badge badge-secondary align-middle ml-2">9</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Alex-H-S-Chik/ class=align-middle data-uni=uni0433>Alex H. S. Chik</a>
<span class="badge badge-secondary align-middle ml-2">8</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Melissa-B-Glier/ class=align-middle data-uni=uni0518>Melissa B. Glier</a>
<span class="badge badge-secondary align-middle ml-2">8</span></li><li class=list-group-item><a href=/gwf-publications/people/X/Xiaoli-Pang/ class=align-middle data-uni=uni0015>Xiaoli Pang</a>
<span class="badge badge-secondary align-middle ml-2">8</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yuanyuan-Qiu/ class=align-middle data-uni=uni0432>Yuanyuan Qiu</a>
<span class="badge badge-secondary align-middle ml-2">8</span></li><li class=list-group-item><a href=/gwf-publications/people/P/Patrick-M-DAoust/ class=align-middle data-uni=uni0028>Patrick M. D’Aoust</a>
<span class="badge badge-secondary align-middle ml-2">8</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Natalie-Prystajecky/ class=align-middle data-uni=uni0013>Natalie Prystajecky</a>
<span class="badge badge-secondary align-middle ml-2">8</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Meghan-Fuzzen/ class=align-middle data-uni=uni0002>Meghan Fuzzen</a>
<span class="badge badge-secondary align-middle ml-2">7</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yuwei-Xie/ class=align-middle data-uni=uni0056>Yuwei Xie</a>
<span class="badge badge-secondary align-middle ml-2">6</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Minqing-Ivy-Yang/ class=align-middle data-uni=uni0031>Minqing Ivy Yang</a>
<span class="badge badge-secondary align-middle ml-2">6</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jianxian-Sun/ class=align-middle data-uni=uni0031>Jianxian Sun</a>
<span class="badge badge-secondary align-middle ml-2">6</span></li><li class=list-group-item><a href=/gwf-publications/people/E/Elizabeth-A-Edwards/ class=align-middle data-uni=uni0031>Elizabeth A. Edwards</a>
<span class="badge badge-secondary align-middle ml-2">6</span></li><li class=list-group-item><a href=/gwf-publications/people/T/Tyson-E-Graber/ class=align-middle data-uni=uni0114>Tyson E. Graber</a>
<span class="badge badge-secondary align-middle ml-2">6</span></li><li class=list-group-item><a href=/gwf-publications/people/R/R-Michael-L-McKay/ class=align-middle data-uni=uni0169>R. Michael L. McKay</a>
<span class="badge badge-secondary align-middle ml-2">5</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Nathanael-BJ-Harper/ class=align-middle data-uni=uni0002>Nathanael B.J. Harper</a>
<span class="badge badge-secondary align-middle ml-2">5</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Samina-Hayat/ class=align-middle data-uni=uni0002>Samina Hayat</a>
<span class="badge badge-secondary align-middle ml-2">5</span></li><li class=list-group-item><a href=/gwf-publications/people/J/JeanBaptiste-Burnet/ class=align-middle data-uni=uni0190>Jean‐Baptiste Burnet</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Sarah-Dorner/ class=align-middle data-uni=uni0190>Sarah Dorner</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/Q/Qiudi-Geng/ class=align-middle data-uni=uni0169>Qiudi Geng</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Michael-R-Mulvey/ class=align-middle data-uni=uni0014>Michael R. Mulvey</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/B/Bernadette-Conant/ class=align-middle data-uni=uni0935>Bernadette Conant</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Steve-E-Hrudey/ class=align-middle data-uni=uni0015>Steve E. Hrudey</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/L/Leslie-M-Bragg/ class=align-middle data-uni=uni0002>Leslie M. Bragg</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/P/Philip-J-Schmidt/ class=align-middle data-uni=uni0002>Philip J. Schmidt</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Nicole-Acosta/ class=align-middle data-uni=uni0007>Nicole Acosta</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Casey-R-J-Hubert/ class=align-middle data-uni=uni0007>Casey R. J. Hubert</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jennifer-Kopetzky/ class=align-middle data-uni=uni1060>Jennifer Kopetzky</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Monica-B-Emelko/ class=align-middle data-uni=uni0002>Monica B. Emelko</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Chrystal-Landgraff/ class=align-middle data-uni=uni0104>Chrystal Landgraff</a>
<span class="badge badge-secondary align-middle ml-2">3</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jonathan-K-Challis/ class=align-middle data-uni=uni0034>Jonathan K. Challis</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/F/Femi-F-Oloye/ class=align-middle data-uni=uni0226>Femi F. Oloye</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Mohsen-Asadi/ class=align-middle data-uni=uni0001>Mohsen Asadi</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jenna-Cantin/ class=align-middle data-uni=uni0001>Jenna Cantin</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Markus-Brinkmann/ class=align-middle data-uni=uni0001>Markus Brinkmann</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/K/Kerry-N-McPhedran/ class=align-middle data-uni=uni0001>Kerry N. McPhedran</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Mike-Sadowski/ class=align-middle data-uni=uni1027>Mike Sadowski</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/P/Paul-D-Jones/ class=align-middle data-uni=uni0001>Paul D. Jones</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Authors-presented-in-alphabetical-order/ class=align-middle data-uni=uni0000>Authors presented in alphabetical order</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/E/Eric-J-Arts/ class=align-middle data-uni=uni0035>Eric J. Arts</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/R/R-Stephen-Brown/ class=align-middle data-uni=uni0049>R. Stephen Brown</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/D/David-Bulir/ class=align-middle data-uni=uni0004>David Bulir</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/T/Trevor-C-Charles/ class=align-middle data-uni=uni1437>Trevor C. Charles</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Christopher-T-DeGroot/ class=align-middle data-uni=uni0035>Christopher T. DeGroot</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/JeanPaul-Desaulniers/ class=align-middle data-uni=uni0448>Jean‐Paul Desaulniers</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/K/Kimberley-Gilbride/ class=align-middle data-uni=uni0163>Kimberley Gilbride</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jodi-Gilchrist/ class=align-middle data-uni=uni1438>Jodi Gilchrist</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/L/Lawrence-Goodridge/ class=align-middle data-uni=uni0011>Lawrence Goodridge</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Marc-Habash/ class=align-middle data-uni=uni1007>Marc Habash</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/P/Peter-J%C3%BCni/ class=align-middle data-uni=uni0085>Peter Jüni</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Andrea-E-Kirkwood/ class=align-middle data-uni=uni0448>Andrea E. Kirkwood</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/James-Knockleby/ class=align-middle data-uni=uni0449>James Knockleby</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Christopher-J-Kyle/ class=align-middle data-uni=uni0074>Christopher J. Kyle</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/D/Douglas-G-Manuel/ class=align-middle data-uni=uni0028>Douglas G. Manuel</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/E/Edgard-M-Mejia/ class=align-middle data-uni=uni0014>Edgard M. Mejia</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Aleksandra-Mloszewska/ class=align-middle data-uni=uni0449>Aleksandra Mloszewska</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/B/Banu-%C3%96rmeci/ class=align-middle data-uni=uni0069>Banu Örmeci</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Claire-Oswald/ class=align-middle data-uni=uni0163>Claire Oswald</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Sarah-Jane-Payne/ class=align-middle data-uni=uni0049>Sarah Jane Payne</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/H/Hui-Peng/ class=align-middle data-uni=uni0031>Hui Peng</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Art-F-Y-Poon/ class=align-middle data-uni=uni0035>Art F. Y. Poon</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/D/Denina-Simmons/ class=align-middle data-uni=uni0448>Denina Simmons</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/G/Gustavo-Ybazeta/ class=align-middle data-uni=uni0449>Gustavo Ybazeta</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/N/Natacha-S-Hogan/ class=align-middle data-uni=uni0001>Natacha S. Hogan</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/K/Kevin-Kristian/ class=align-middle data-uni=uni1516>Kevin Kristian</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Mark-P-Keller/ class=align-middle data-uni=uni0046>Mark P. Keller</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li></div></ul></div></div><div class="col-12 col-md-6 col-lg-12"><div class="card my-2 my-md-0 my-lg-2"><h5 class=card-header>Venues</h5><ul class="list-group list-group-flush list-group-compact"><li class=list-group-item><a href=/gwf-publications/venues/gwf/ class=align-middle>GWF</a><span class="badge badge-secondary align-middle ml-2">10</span></li></ul></div></div></div></div></div></section></div><footer class="bg-gradient-light py-2 py-xl-3 mt-3 mt-md-4 mt-xl-5"><div class=container><p class="text-muted small px-1">Global Water Futures Publications!</p></div></footer><script src=https://code.jquery.com/jquery-3.3.1.slim.min.js integrity=sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js integrity=sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut crossorigin=anonymous></script><script src=https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js integrity=sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k crossorigin=anonymous></script><script>$(function(){$('[data-toggle="tooltip"]').tooltip(),$("#toggle-all-abstracts")&&($("#toggle-all-abstracts").click(function(){var e=$("#toggle-all-abstracts");e.attr("disabled",!0),e.attr("data-toggle-state")=="hide"?($(".abstract-collapse").collapse("show"),e.attr("data-toggle-state","show")):($(".abstract-collapse").collapse("hide"),e.attr("data-toggle-state","hide")),e.attr("disabled",!1)}),$("#toggle-all-abstracts").attr("disabled",!1))})</script></body></html>