<!doctype html><html lang=en-us><head><link rel=preconnect href=https://fonts.googleapis.com><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@700&display=swap" rel=stylesheet><meta charset=utf-8><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><script src=https://ajax.googleapis.com/ajax/libs/jquery/3.6.1/jquery.min.js></script><script>window.onload=function(){document.getElementById("expand-proj").addEventListener("click",function(){this.textContent=="show less..."?this.textContent="show all...":this.textContent="show less..."}),document.getElementById("expand-uni").addEventListener("click",function(){this.textContent=="show less..."?this.textContent="show all...":this.textContent="show less..."})}</script><script>var unis={};$(document).on("change","input[type='checkbox']",function(){if(this.id.startsWith("check_year")){var e,n,t=$(':input[id^="check_year"]').map(function(){return $(this).prop("checked")}).get().includes(!0);const s=document.getElementById("main").children;if($("#year_list li:first-child").css("display")=="none"&&($("#year_list li:first-child").prop("style","display:initial !important"),$('div[id^="year_"]').prop("style","display:none !important")),t)for(child of s)child.id.startsWith(this.id.slice(-9))&&(this.checked?child.style.display="initial":child.style.display="none");else{$("#year_list li:first-child").prop("style","display:none !important");for(child of s)child.style.display="initial"}}else if(this.id.startsWith("uni"))if(console.log("Hi2"),t=$(':input[id^="uni"]').map(function(){return $(this).prop("checked")}).get().includes(!0),e=this.id.slice(-7),unis[e]==void 0&&(unis[e]=!0),this.checked?unis[e]=!0:unis[e]=!1,$("#university_list li:first-child").css("display")=="none"&&($("#university_list li:first-child").prop("style","display:initial !important"),$('div[data-prj^="prj"]').each(function(){$(this).children().first().prop("style","display:none !important")})),t){const t=document.querySelectorAll("[data-uni="+e+"]");for(child of t)if(this.checked)child.parentNode.parentNode.setAttribute("style","display:flex !important");else{disp_off_flag=!0;for(child of child.parentNode.querySelectorAll("[data-uni]"))if(unis[child.getAttribute("data-uni")]==void 0&&(unis[child.getAttribute("data-uni")]=!0),unis[child.getAttribute("data-uni")]==!0){disp_off_flag=!1;break}disp_off_flag&&child.parentNode.parentNode.setAttribute("style","display:none !important")}}else $("#university_list li:first-child").prop("style","display:none !important"),$('div[data-prj^="prj"]').each(function(){$(this).children().first().prop("style","display:flex !important")});else if(this.id.startsWith("prj"))if(t=$(':input[id^="prj"]').map(function(){return $(this).prop("checked")}).get().includes(!0),n=this.id.slice(4),$("#project_list li:first-child").css("display")=="none"&&($("#project_list li:first-child").prop("style","display:initial !important"),$('div[data-prj^="prj"]').each(function(){$(this).prop("style","display:none !important")})),t){const e=document.querySelectorAll("[data-prj="+n+"]");for(child of e)this.checked?child.setAttribute("style","display:initial !important"):child.setAttribute("style","display:none !important")}else console.log("all unckecked"),$("#project_list li:first-child").prop("style","display:none !important"),$('div[data-prj^="prj"]').each(function(){$(this).prop("style","display:initial !important")})});function clearAll(e){e=="year"?($(':input[id^="check_year"]').prop("checked",!1),$('div[id^="year_"]').prop("style","display:initial;"),$("#year_list li:first-child").prop("style","display:none !important")):e=="uni"?($(':input[id^="uni"]').prop("checked",!1),$('div[data-prj^="prj"]').each(function(){$(this).children().first().prop("style","display:flex !important")}),$("#university_list li:first-child").prop("style","display:none !important")):e=="prj"&&($(':input[id^="prj"]').prop("checked",!1),$('div[data-prj^="prj"]').each(function(){$(this).prop("style","display:initial !important")}),$("#project_list li:first-child").prop("style","display:none !important"))}</script><style>.arrowup{border:solid #fff;border-width:0 3px 3px 0;display:inline-block;padding:5px;transform:rotate(-135deg);-webkit-transform:rotate(-135deg)}#topBtn{display:none;position:fixed;bottom:20px;right:30px;z-index:99;font-size:30px;border:none;outline:none;background-color:#29abe0;color:#fff;cursor:pointer;padding:15px;border-radius:8px;height:10dp}#topBtn:hover{background-color:#1b8dbb}.card-header-custom{cursor:pointer;-webkit-user-select:none;-ms-user-select:none;user-select:none}.clear-widget{color:#747474!important;background-color:#dee2e6;font-size:80%!important}.clear-widget:hover{color:#212529!important;background-color:#c1c9d0;cursor:pointer}</style><!--[if IEMobile]><meta http-equiv=cleartype content="on"><![endif]--><title>Juewen Liu - GWF Publications</title>
<meta name=generator content="Hugo 0.140.2"><link href=/gwf-publications/gwficon.ico rel="shortcut icon" type=image/x-icon><link rel=stylesheet href=/gwf-publications/css/main.min.b1d14a9a8f6bb9c608ca4de9aad72a6e06945119f97951f2908522dc220e6277.css media=screen><link rel=stylesheet href=https://use.fontawesome.com/releases/v5.7.2/css/all.css integrity=sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr crossorigin=anonymous><link rel=stylesheet href=/css/academicons.min.css></head><body><header style='display:block;box-sizing:border-box;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;line-height:1.5;color:#222;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%'><div id=navbar-container style='height:54px;width:100%;background:rgba(255,255,255,.95);overflow:hidden;-webkit-box-shadow:rgba(0,0,0,9%)0 1px 0;box-shadow:rgba(0,0,0,9%)0 1px;box-sizing:border-box;display:block;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;line-height:1.5;color:#222;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%'><a href=/ style='color:#0072bc;text-decoration:none;display:inline-block;height:100%;line-height:54px;border-right:1px solid #e6e6e6;margin-left:6px;padding-right:6px;outline-offset:-4px!important;font-weight:700;background:0 0;box-sizing:border-box;cursor:pointer;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;-webkit-tap-highlight-color:transparent'><img src=/gwf-publications/images/gwf-logo.png alt="GWF Logo" style='margin-right:0;height:54px;padding:6px;width:auto;margin-top:-2px;max-width:100%;vertical-align:middle;border:0;box-sizing:border-box;color:#0072bc;line-height:54px;font-weight:700;cursor:pointer;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;font-size:16px;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%'>
</a><a href=/ style='text-decoration:none;color:#222!important;display:inline-block;font-size:16px;line-height:54px;font-weight:700;outline-offset:-4px!important;padding:0 6px;margin:0 -6px;background:0 0;box-sizing:border-box;cursor:pointer;font-family:open sans,helvetica neue,Helvetica,Arial,sans-serif;-webkit-tap-highlight-color:transparent;-webkit-text-size-adjust:100%;padding-left:4px'>Global Water Futures</a></div></header><div id=main-container class=container><section id=main><h2 id=title><span class=font-weight-normal>Juewen</span> <span class=font-weight-bold>Liu</span></h2><hr><div class=row><div class=col-lg-9><h4>2023</h4><div data-prj=prj10><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1002/anie.202212879 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G23-37001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G23-37001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G23-37001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G23-37001/>Simultaneous Detection of L‐Lactate and D‐Glucose Using DNA Aptamers in Human Blood Serum**</a></strong><br><a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a>,
<a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G23-37/ class=text-muted>Angewandte Chemie International Edition, Volume 62, Issue 12</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G23-37001><div class="card-body p-3 small">L-lactate is a key metabolite indicative of physiological states, glycolysis pathways, and various diseases such as sepsis, heart attack, lactate acidosis, and cancer. Detection of lactate has been relying on a few enzymes that need additional oxidants. In this work, DNA aptamers for L-lactate were obtained using a library-immobilization selection method and the highest affinity aptamer reached a Kd of 0.43 mM as determined using isothermal titration calorimetry. The aptamers showed up to 50-fold selectivity for L-lactate over D-lactate and had little responses to other closely related analogs such as pyruvate or 3-hydroxybutyrate. A fluorescent biosensor based on the strand displacement method showed a limit of detection of 0.55 mM L-lactate, and the sensor worked in 90 % serum. Simultaneous detection of L-lactate and D-glucose in the same solution was achieved. This work has broadened the scope of aptamers to simple metabolites and provided a useful probe for continuous and multiplexed monitoring.</div></div><div data-prj=prj10><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1002/anie.202212879 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G23-37001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G23-37001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G23-37001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G23-37001/>Simultaneous Detection of L‐Lactate and D‐Glucose Using DNA Aptamers in Human Blood Serum**</a></strong><br><a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a>,
<a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G23-37/ class=text-muted>Angewandte Chemie International Edition, Volume 62, Issue 12</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G23-37001><div class="card-body p-3 small">L-lactate is a key metabolite indicative of physiological states, glycolysis pathways, and various diseases such as sepsis, heart attack, lactate acidosis, and cancer. Detection of lactate has been relying on a few enzymes that need additional oxidants. In this work, DNA aptamers for L-lactate were obtained using a library-immobilization selection method and the highest affinity aptamer reached a Kd of 0.43 mM as determined using isothermal titration calorimetry. The aptamers showed up to 50-fold selectivity for L-lactate over D-lactate and had little responses to other closely related analogs such as pyruvate or 3-hydroxybutyrate. A fluorescent biosensor based on the strand displacement method showed a limit of detection of 0.55 mM L-lactate, and the sensor worked in 90 % serum. Simultaneous detection of L-lactate and D-glucose in the same solution was achieved. This work has broadened the scope of aptamers to simple metabolites and provided a useful probe for continuous and multiplexed monitoring.</div></div><div data-prj=prj31><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1002/exp.20230018 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G23-135001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G23-135001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G23-135001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G23-135001/>Catalytic and biocatalytic degradation of microplastics</a></strong><br><a href=/gwf-publications/people/M/Mohamad-Zandieh/ data-uni=uni0002>Mohamad Zandieh</a>,
<a href=/gwf-publications/people/E/Erin-Griffiths/ data-uni=uni0002>Erin Griffiths</a>,
<a href=/gwf-publications/people/A/Alexander-Waldie/ data-uni=uni0002>Alexander Waldie</a>,
<a href=/gwf-publications/people/S/Shuhuan-Li/ data-uni=uni0161>Shuhuan Li</a>,
<a href=/gwf-publications/people/J/John-F-Honek/ data-uni=uni0002>John F. Honek</a>,
<a href=/gwf-publications/people/F/Fereidoun-Rezanezhad/ data-uni=uni0002>Fereidoun Rezanezhad</a>,
<a href=/gwf-publications/people/P/Philippe-Van-Cappellen/ data-uni=uni0002>Philippe Van Cappellen</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a>,
<a href=/gwf-publications/people/M/Mohamad-Zandieh/ data-uni=uni0002>Mohamad Zandieh</a>,
<a href=/gwf-publications/people/E/Erin-Griffiths/ data-uni=uni0002>Erin Griffiths</a>,
<a href=/gwf-publications/people/A/Alexander-Waldie/ data-uni=uni0002>Alexander Waldie</a>,
<a href=/gwf-publications/people/S/Shuhuan-Li/ data-uni=uni0161>Shuhuan Li</a>,
<a href=/gwf-publications/people/J/John-F-Honek/ data-uni=uni0002>John F. Honek</a>,
<a href=/gwf-publications/people/F/Fereidoun-Rezanezhad/ data-uni=uni0002>Fereidoun Rezanezhad</a>,
<a href=/gwf-publications/people/P/Philippe-Van-Cappellen/ data-uni=uni0002>Philippe Van Cappellen</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G23-135/ class=text-muted>Exploration, Volume 4, Issue 3</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G23-135001><div class="card-body p-3 small">In recent years, there has been a surge in annual plastic production, which has contributed to growing environmental challenges, particularly in the form of microplastics. Effective management of plastic and microplastic waste has become a critical concern, necessitating innovative strategies to address its impact on ecosystems and human health. In this context, catalytic degradation of microplastics emerges as a pivotal approach that holds significant promise for mitigating the persistent effects of plastic pollution. In this article, we critically explored the current state of catalytic degradation of microplastics and discussed the definition of degradation, characterization methods for degradation products, and the criteria for standard sample preparation. Moreover, the significance and effectiveness of various catalytic entities, including enzymes, transition metal ions (for the Fenton reaction), nanozymes, and microorganisms are summarized. Finally, a few key issues and future perspectives regarding the catalytic degradation of microplastics are proposed.</div></div><div data-prj=prj31><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1002/exp.20230018 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G23-135001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G23-135001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G23-135001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G23-135001/>Catalytic and biocatalytic degradation of microplastics</a></strong><br><a href=/gwf-publications/people/M/Mohamad-Zandieh/ data-uni=uni0002>Mohamad Zandieh</a>,
<a href=/gwf-publications/people/E/Erin-Griffiths/ data-uni=uni0002>Erin Griffiths</a>,
<a href=/gwf-publications/people/A/Alexander-Waldie/ data-uni=uni0002>Alexander Waldie</a>,
<a href=/gwf-publications/people/S/Shuhuan-Li/ data-uni=uni0161>Shuhuan Li</a>,
<a href=/gwf-publications/people/J/John-F-Honek/ data-uni=uni0002>John F. Honek</a>,
<a href=/gwf-publications/people/F/Fereidoun-Rezanezhad/ data-uni=uni0002>Fereidoun Rezanezhad</a>,
<a href=/gwf-publications/people/P/Philippe-Van-Cappellen/ data-uni=uni0002>Philippe Van Cappellen</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a>,
<a href=/gwf-publications/people/M/Mohamad-Zandieh/ data-uni=uni0002>Mohamad Zandieh</a>,
<a href=/gwf-publications/people/E/Erin-Griffiths/ data-uni=uni0002>Erin Griffiths</a>,
<a href=/gwf-publications/people/A/Alexander-Waldie/ data-uni=uni0002>Alexander Waldie</a>,
<a href=/gwf-publications/people/S/Shuhuan-Li/ data-uni=uni0161>Shuhuan Li</a>,
<a href=/gwf-publications/people/J/John-F-Honek/ data-uni=uni0002>John F. Honek</a>,
<a href=/gwf-publications/people/F/Fereidoun-Rezanezhad/ data-uni=uni0002>Fereidoun Rezanezhad</a>,
<a href=/gwf-publications/people/P/Philippe-Van-Cappellen/ data-uni=uni0002>Philippe Van Cappellen</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G23-135/ class=text-muted>Exploration, Volume 4, Issue 3</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G23-135001><div class="card-body p-3 small">In recent years, there has been a surge in annual plastic production, which has contributed to growing environmental challenges, particularly in the form of microplastics. Effective management of plastic and microplastic waste has become a critical concern, necessitating innovative strategies to address its impact on ecosystems and human health. In this context, catalytic degradation of microplastics emerges as a pivotal approach that holds significant promise for mitigating the persistent effects of plastic pollution. In this article, we critically explored the current state of catalytic degradation of microplastics and discussed the definition of degradation, characterization methods for degradation products, and the criteria for standard sample preparation. Moreover, the significance and effectiveness of various catalytic entities, including enzymes, transition metal ions (for the Fenton reaction), nanozymes, and microorganisms are summarized. Finally, a few key issues and future perspectives regarding the catalytic degradation of microplastics are proposed.</div></div><h4>2022</h4><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1021/acs.analchem.1c04349 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G22-64001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G22-64001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G22-64001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G22-64001/>Selection of Aptamers for Sensing Caffeine and Discrimination of Its Three Single Demethylated Analogues</a></strong><br><a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G22-64/ class=text-muted>Analytical Chemistry, Volume 94, Issue 7</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G22-64001><div class="card-body p-3 small">With the growing consumption of caffeine-containing beverages, detection of caffeine has become an important biomedical, bioanalytical, and environmental topic. We herein isolated four high-quality aptamers for caffeine with dissociation constants ranging from 2.2 to 14.6 μM as characterized using isothermal titration calorimetry. Different binding patterns were obtained for the three single demethylated analogues: theobromine, theophylline, and paraxanthine, highlighting the effect of the molecular symmetry of the arrangement of the three methyl groups in caffeine. A structure-switching fluorescent sensor was designed showing a detection limit of 1.2 μM caffeine, which reflected the labeled caffeine concentration within 6.1% difference for eight commercial beverages. In 20% human serum, a detection limit of 4.0 μM caffeine was achieved. With the four aptamer sensors forming an array, caffeine and the three analogues were well separated from nine other closely related molecules.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1016/j.trac.2021.116480 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G22-67001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G22-67001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G22-67001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G22-67001/>Critical evaluation of aptamer binding for biosensor designs</a></strong><br><a href=/gwf-publications/people/Y/Yichen-Zhao/ data-uni=uni0002>Yichen Zhao</a>,
<a href=/gwf-publications/people/K/Kayvan-Yavari/ data-uni=uni0002>Kayvan Yavari</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G22-67/ class=text-muted>TrAC Trends in Analytical Chemistry, Volume 146</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G22-67001><div class="card-body p-3 small">Over the last three decades, numerous aptamer-based biosensors have been reported. The basis of these sensors is the selective binding of target analytes by aptamers. In the last few years, a number of papers have been published questioning the binding ability of some popular aptamers such as those documented for As(III), ampicillin, chloramphenicol, isocarbophos, phorate and dopamine. In this article, these papers are reviewed, and the binding assays are described, which may provide possible reasons for obtaining false positive aptamers. Additionally, relevant aptamer selection methods and typical characterization steps are described. It is found that for small molecular targets, using an immobilized library might result in better aptamers. Furthermore, the importance of carefully designed controls to ensure the quality of binding assays is discussed, especially in the case of mutated nonbinding aptamers. Only then, with fully validated aptamers, can subsequent biosensor design bring about meaningful results. • The first critical review of the literature on aptamers that were proven to be non-binding sequences. • Five different aptamers for various small molecules reviewed. • Possible reasons for the generation of such non-binding aptamer sequences proposed and methods to avoid them described.</div></div><div data-prj=prj10><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1016/j.trac.2022.116639 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G22-156001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G22-156001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G22-156001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G22-156001/>Deployment of functional DNA-based biosensors for environmental water analysis</a></strong><br><a href=/gwf-publications/people/Y/Yichen-Zhao/ data-uni=uni0002>Yichen Zhao</a>,
<a href=/gwf-publications/people/K/Kayvan-Yavari/ data-uni=uni0002>Kayvan Yavari</a>,
<a href=/gwf-publications/people/Y/Yihao-Wang/ data-uni=uni0002>Yihao Wang</a>,
<a href=/gwf-publications/people/K/Kunfu-Pi/ data-uni=uni0174>Kunfu Pi</a>,
<a href=/gwf-publications/people/P/Philippe-Van-Cappellen/ data-uni=uni0002>Philippe Van Cappellen</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G22-156/ class=text-muted>TrAC Trends in Analytical Chemistry, Volume 153</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G22-156001><div class="card-body p-3 small">Various functional DNA molecules have been used for the detection of environmental contaminants in water, but their practical applications have been limited. To address this gap, this review highlights the efforts to develop field-deployable water quality biosensors. The biosensor devices include microfluidic, lateral flow and paper-based devices, and other novel ideas such as the conversion of glucometers for the detection of environmental analytes. In addition, we also review DNA-functionalized hydrogels and their use in diffusive gradients in thin films (DGT) devices. We classify the sensors into one-step and two-step assays and discuss their practical implications. While the review is focused on works reported in the last five years, some classic early works are cited as well. Overall, most of the existing work only tested spiked water samples. Future work needs to shift to real environmental samples and the comparison of DNA-based sensors to standard analytical methods. • Recent development in field-deployable functional DNA based biosensors for environmental water monitoring reviewed. • Articulated the concept of one-step and two-step assays. • Microfluidic device, lateral flow device, paper, hydrogel, and glucose meter based examples reviewed.</div></div><div data-prj=prj10><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1021/acschembio.2c00179 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G22-163001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G22-163001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G22-163001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G22-163001/>A DNA Aptamer for Theophylline with Ultrahigh Selectivity Reminiscent of the Classic RNA Aptamer</a></strong><br><a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G22-163/ class=text-muted>ACS Chemical Biology, Volume 17, Issue 8</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G22-163001><div class="card-body p-3 small">Since the report of the RNA aptamer for theophylline, theophylline has become a key molecule in chemical biology for designing RNA switches and riboswitches. In addition, theophylline is an important drug for treating airway diseases including asthma. The classic RNA aptamer with excellent selectivity for theophylline has been used to design biosensors, although DNA aptamers are more desirable for stability and cost considerations. In this work, we selected DNA aptamers for theophylline, and all the top sequences shared the same binding motifs. Binding was confirmed using isothermal titration calorimetry and a nuclease digestion assay, showing a dissociation constant (Kd) around 0.5 μM theophylline. The Theo2201 aptamer can be truncated down to 23-mer while still has a Kd of 9.8 μM. The selectivity for theophylline over caffeine is around 250,000-fold based on a strand-displacement assay, which was more than 20-fold higher compared to the classic RNA aptamer. For other tested analogs, the DNA aptamer also showed better selectivity. Using the structure-switching aptamer sensor design method, a detection limit of 17 nM theophylline was achieved in the selection buffer, and a detection limit of 31 nM was obtained in 10% serum.</div></div><h4>2021</h4><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1016/j.teac.2021.e00143 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-119001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-119001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-119001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-119001/>Critical review of bio/nano sensors for arsenic detection</a></strong><br><a href=/gwf-publications/people/C/Chenghua-Zong/ data-uni=uni0353>Chenghua Zong</a>,
<a href=/gwf-publications/people/X/Xiaoting-Jin/ data-uni=uni0831>Xiaoting Jin</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a>,
<a href=/gwf-publications/people/C/Chenghua-Zong/ data-uni=uni0353>Chenghua Zong</a>,
<a href=/gwf-publications/people/X/Xiaoting-Jin/ data-uni=uni0831>Xiaoting Jin</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G21-119/ class=text-muted>Trends in Environmental Analytical Chemistry, Volume 32</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-119001><div class="card-body p-3 small">Detection of arsenic is a long-standing challenge in environmental analytical chemistry. In recent years, using biomolecules and nanomaterials for sensing arsenic has been growingly reported. In this article, this field is critically reviewed based on some recent fundamental understandings including interactions between arsenic and gold, thiol, and DNA aptamers. First, taking advantage of the adsorption of As(III) on noble metal surfaces such as silver and gold, sensors were developed based on surface enhanced Raman spectroscopy, electrochemistry and colorimetry. In addition, by functionalizing metal nanoparticles with thiol containing molecules, As(III) induced aggregation of the particles based on As(III)/thiol interactions. As(V) interacts with metal oxides strongly and competitive sensors were developed by displacing pre-adsorbed DNA oligonucleotides. A DNA aptamer was selected for As(III) and many sensors were reported based on this aptamer, although careful binding measurements indicated that the sequence has no affinity towards As(III). Overall, bio/nano systems are promising for the detection of arsenic. Future work on fundamental studies, searching for more specific arsenic binding materials and aptamers, incorporation of sensors into portable devices, and more systematic test of sensors in real samples could be interesting and useful research topics.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1016/j.teac.2021.e00143 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-119001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-119001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-119001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-119001/>Critical review of bio/nano sensors for arsenic detection</a></strong><br><a href=/gwf-publications/people/C/Chenghua-Zong/ data-uni=uni0353>Chenghua Zong</a>,
<a href=/gwf-publications/people/X/Xiaoting-Jin/ data-uni=uni0831>Xiaoting Jin</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a>,
<a href=/gwf-publications/people/C/Chenghua-Zong/ data-uni=uni0353>Chenghua Zong</a>,
<a href=/gwf-publications/people/X/Xiaoting-Jin/ data-uni=uni0831>Xiaoting Jin</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G21-119/ class=text-muted>Trends in Environmental Analytical Chemistry, Volume 32</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-119001><div class="card-body p-3 small">Detection of arsenic is a long-standing challenge in environmental analytical chemistry. In recent years, using biomolecules and nanomaterials for sensing arsenic has been growingly reported. In this article, this field is critically reviewed based on some recent fundamental understandings including interactions between arsenic and gold, thiol, and DNA aptamers. First, taking advantage of the adsorption of As(III) on noble metal surfaces such as silver and gold, sensors were developed based on surface enhanced Raman spectroscopy, electrochemistry and colorimetry. In addition, by functionalizing metal nanoparticles with thiol containing molecules, As(III) induced aggregation of the particles based on As(III)/thiol interactions. As(V) interacts with metal oxides strongly and competitive sensors were developed by displacing pre-adsorbed DNA oligonucleotides. A DNA aptamer was selected for As(III) and many sensors were reported based on this aptamer, although careful binding measurements indicated that the sequence has no affinity towards As(III). Overall, bio/nano systems are promising for the detection of arsenic. Future work on fundamental studies, searching for more specific arsenic binding materials and aptamers, incorporation of sensors into portable devices, and more systematic test of sensors in real samples could be interesting and useful research topics.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1039/d1cc03102c data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-124001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-124001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-124001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-124001/>Selection of a self-cleaving ribozyme activated in a chemically and thermally denaturing environment</a></strong><br><a href=/gwf-publications/people/L/Lingzi-Ma/ data-uni=uni0002>Lingzi Ma</a>,
<a href=/gwf-publications/people/Z/Zhicheng-Huang/ data-uni=uni0272>Zhicheng Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a>,
<a href=/gwf-publications/people/L/Lingzi-Ma/ data-uni=uni0002>Lingzi Ma</a>,
<a href=/gwf-publications/people/Z/Zhicheng-Huang/ data-uni=uni0272>Zhicheng Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G21-124/ class=text-muted>Chemical Communications, Volume 57, Issue 62</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-124001><div class="card-body p-3 small">A self-cleaving ribozyme was obtained from &lt;italic>in vitro&lt;/italic> selection, displaying site-specific cleavage activity under various denaturing conditions, such as high temperatures, 20 M formamide, and low salt concentrations.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1039/d1cc03102c data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-124001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-124001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-124001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-124001/>Selection of a self-cleaving ribozyme activated in a chemically and thermally denaturing environment</a></strong><br><a href=/gwf-publications/people/L/Lingzi-Ma/ data-uni=uni0002>Lingzi Ma</a>,
<a href=/gwf-publications/people/Z/Zhicheng-Huang/ data-uni=uni0272>Zhicheng Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a>,
<a href=/gwf-publications/people/L/Lingzi-Ma/ data-uni=uni0002>Lingzi Ma</a>,
<a href=/gwf-publications/people/Z/Zhicheng-Huang/ data-uni=uni0272>Zhicheng Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G21-124/ class=text-muted>Chemical Communications, Volume 57, Issue 62</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-124001><div class="card-body p-3 small">A self-cleaving ribozyme was obtained from &lt;italic>in vitro&lt;/italic> selection, displaying site-specific cleavage activity under various denaturing conditions, such as high temperatures, 20 M formamide, and low salt concentrations.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1016/j.ccr.2020.213624 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-126001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-126001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-126001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-126001/>Phosphorothioate nucleic acids for probing metal binding, biosensing and nanotechnology</a></strong><br><a href=/gwf-publications/people/R/Runjhun-Saran/ data-uni=uni0002>Runjhun Saran</a>,
<a href=/gwf-publications/people/Z/Zhicheng-Huang/ data-uni=uni0272>Zhicheng Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a>,
<a href=/gwf-publications/people/R/Runjhun-Saran/ data-uni=uni0002>Runjhun Saran</a>,
<a href=/gwf-publications/people/Z/Zhicheng-Huang/ data-uni=uni0272>Zhicheng Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G21-126/ class=text-muted>Coordination Chemistry Reviews, Volume 428</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-126001><div class="card-body p-3 small">• Fundamentals of phosphorothioate nucleic acids reviewed from synthesis to metal binding. • Applications of phosphorothioate nucleic acids in developing biosensors and chemical biology reviewed. • Applications of phosphorothioate DNA in assembly and the directed growth of nanomaterials reviewed. Phosphorothioate (PS) modification replaces one of the non-bridging oxygen atoms by sulfur in the phosphate backbone of nucleic acids. While PS DNAs have been traditionally used as nuclease-resistant antisense agents and PS RNA as probe of metal binding in ribozymes, multiple new applications have emerged in recent years. In this review, we start by briefly introducing the structure and synthesis of PS nucleic acids followed by their fundamental chemical and biochemical properties. Further, their recently emerged surface science applications are discussed, such as attachment of DNA to various surfaces and nanomaterials containing thiophilic metals such as gold, silver and cadmium, and templating the growth of these materials. Their role in conferring structural effects in the presence of certain metal ions and in fishing out novel aptamers are also discussed. Covalent chemistry can be performed on the sulfur atom for further grafting functional groups to the backbone of DNA. For PS RNA, we discuss their role as probes for metal binding in ribozymes and DNAzymes, which leads to applications in detection of thiophilic metal ions. Since each PS modification site produces a chiral phosphorus center, the synthesis and purification of diastereomers and their applications are emphasized throughout this review. In the end, a few future research directions are discussed.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1016/j.ccr.2020.213624 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-126001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-126001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-126001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-126001/>Phosphorothioate nucleic acids for probing metal binding, biosensing and nanotechnology</a></strong><br><a href=/gwf-publications/people/R/Runjhun-Saran/ data-uni=uni0002>Runjhun Saran</a>,
<a href=/gwf-publications/people/Z/Zhicheng-Huang/ data-uni=uni0272>Zhicheng Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a>,
<a href=/gwf-publications/people/R/Runjhun-Saran/ data-uni=uni0002>Runjhun Saran</a>,
<a href=/gwf-publications/people/Z/Zhicheng-Huang/ data-uni=uni0272>Zhicheng Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G21-126/ class=text-muted>Coordination Chemistry Reviews, Volume 428</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-126001><div class="card-body p-3 small">• Fundamentals of phosphorothioate nucleic acids reviewed from synthesis to metal binding. • Applications of phosphorothioate nucleic acids in developing biosensors and chemical biology reviewed. • Applications of phosphorothioate DNA in assembly and the directed growth of nanomaterials reviewed. Phosphorothioate (PS) modification replaces one of the non-bridging oxygen atoms by sulfur in the phosphate backbone of nucleic acids. While PS DNAs have been traditionally used as nuclease-resistant antisense agents and PS RNA as probe of metal binding in ribozymes, multiple new applications have emerged in recent years. In this review, we start by briefly introducing the structure and synthesis of PS nucleic acids followed by their fundamental chemical and biochemical properties. Further, their recently emerged surface science applications are discussed, such as attachment of DNA to various surfaces and nanomaterials containing thiophilic metals such as gold, silver and cadmium, and templating the growth of these materials. Their role in conferring structural effects in the presence of certain metal ions and in fishing out novel aptamers are also discussed. Covalent chemistry can be performed on the sulfur atom for further grafting functional groups to the backbone of DNA. For PS RNA, we discuss their role as probes for metal binding in ribozymes and DNAzymes, which leads to applications in detection of thiophilic metal ions. Since each PS modification site produces a chiral phosphorus center, the synthesis and purification of diastereomers and their applications are emphasized throughout this review. In the end, a few future research directions are discussed.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1021/acs.biochem.1c00091 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-128001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-128001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-128001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-128001/>Probing Metal-Dependent Phosphate Binding for the Catalysis of the 17E DNAzyme</a></strong><br><a href=/gwf-publications/people/W/Woohyun-J-Moon/ data-uni=uni0354>Woohyun J. Moon</a>,
<a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a>,
<a href=/gwf-publications/people/W/Woohyun-J-Moon/ data-uni=uni0354>Woohyun J. Moon</a>,
<a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G21-128/ class=text-muted>Biochemistry, Volume 60, Issue 24</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-128001><div class="card-body p-3 small">The RNA-cleaving 17E DNAzyme exhibits different levels of cleavage activity in the presence of various divalent metal ions, with Pb2+ giving the fastest cleavage. In this study, the metal-phosphate interaction is probed to understand the trend of activity with different metal ions. For the first-row transition metals, the lowest activity shown by Ni2+ correlates with the inhibition by the inorganic phosphate and its water ligand exchange rate, suggesting inner-sphere metal coordination. Cleavage activity with the two stereoisomers of the phosphorothioate-modified substrates, Rp and Sp, indicated that Mg2+, Mn2+, Fe2+, and Co2+ had the highest Sp:Rp activity ratio of >900. Comparatively, the activity was much less affected using the thiophilic metals, including Pb2+, suggesting inner-sphere coordination. The pH-rate profiles showed that Pb2+ was different than the rest of the metal ions in having a smaller slope and a similar fitted apparent pKa and the pKa of metal-bound water. Combining previous reports and our current results, we propose that Pb2+ most likely plays the role of a general acid while the other metal ions are Lewis acid catalysts interacting with the scissile phosphate.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1021/acs.biochem.1c00091 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-128001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-128001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-128001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-128001/>Probing Metal-Dependent Phosphate Binding for the Catalysis of the 17E DNAzyme</a></strong><br><a href=/gwf-publications/people/W/Woohyun-J-Moon/ data-uni=uni0354>Woohyun J. Moon</a>,
<a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a>,
<a href=/gwf-publications/people/W/Woohyun-J-Moon/ data-uni=uni0354>Woohyun J. Moon</a>,
<a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G21-128/ class=text-muted>Biochemistry, Volume 60, Issue 24</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-128001><div class="card-body p-3 small">The RNA-cleaving 17E DNAzyme exhibits different levels of cleavage activity in the presence of various divalent metal ions, with Pb2+ giving the fastest cleavage. In this study, the metal-phosphate interaction is probed to understand the trend of activity with different metal ions. For the first-row transition metals, the lowest activity shown by Ni2+ correlates with the inhibition by the inorganic phosphate and its water ligand exchange rate, suggesting inner-sphere metal coordination. Cleavage activity with the two stereoisomers of the phosphorothioate-modified substrates, Rp and Sp, indicated that Mg2+, Mn2+, Fe2+, and Co2+ had the highest Sp:Rp activity ratio of >900. Comparatively, the activity was much less affected using the thiophilic metals, including Pb2+, suggesting inner-sphere coordination. The pH-rate profiles showed that Pb2+ was different than the rest of the metal ions in having a smaller slope and a similar fitted apparent pKa and the pKa of metal-bound water. Combining previous reports and our current results, we propose that Pb2+ most likely plays the role of a general acid while the other metal ions are Lewis acid catalysts interacting with the scissile phosphate.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1016/j.aca.2020.12.056 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G21-202001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G21-202001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G21-202001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G21-202001/>Review of recent progress on DNA-based biosensors for Pb2+ detection</a></strong><br><a href=/gwf-publications/people/Y/Yongjie-Yang/ data-uni=uni0926>Yongjie Yang</a>,
<a href=/gwf-publications/people/W/Weixuan-Li/ data-uni=uni0663>Weixuan Li</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G21-202/ class=text-muted>Analytica Chimica Acta, Volume 1147</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G21-202001><div class="card-body p-3 small">Lead (Pb) is a highly toxic heavy metal of great environmental and health concerns, and interestingly Pb 2+ has played important roles in nucleic acids chemistry. Since 2000, using DNA for selective detection of Pb 2+ has become a rapidly growing topic in the analytical community. Pb 2+ can serve as the most active cofactor for RNA-cleaving DNAzymes including the GR5, 17E and 8–17 DNAzymes. Recently, Pb 2+ was found to promote a porphyrin metalation DNAzyme named T30695. In addition, Pb 2+ can tightly bind to various G-quadruplex sequences inducing their unique folding and binding to other molecules such as dyes and hemin. The peroxidase-like activity of G-quadruplex/hemin complexes was also used for Pb 2+ sensing. In this article, these Pb 2+ recognition mechanisms are reviewed from fundamental chemistry to the design of fluorescent, colorimetric, and electrochemical biosensors. In addition, various signal amplification mechanisms such as rolling circle amplification, hairpin hybridization chain reaction and nuclease-assisted methods are coupled to these sensing methods to drive up sensitivity. We mainly cover recent examples published since 2015. In the end, some practical aspects of these sensors and future research opportunities are discussed. • Fundamentals of Pb 2+ recognition by DNA including DNAzymes and G-quadruplex DNA reviewed • Literature in the last five years summarized • Practical aspect of Pb 2+ detection using DNA discussed</div></div><h4>2020</h4><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1002/anie.201915675 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G20-16001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G20-16001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G20-16001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G20-16001/>Target Self‐Enhanced Selectivity in Metal‐Specific DNAzymes</a></strong><br><a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/D/Donatien-de-Rochambeau/ data-uni=uni0057>Donatien de Rochambeau</a>,
<a href=/gwf-publications/people/H/Hanadi-F-Sleiman/ data-uni=uni0057>Hanadi F. Sleiman</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G20-16/ class=text-muted>Angewandte Chemie International Edition, Volume 59, Issue 9</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G20-16001><div class="card-body p-3 small">The more the merrier: Better selectivity for Zn2+ ions is shown for DNAzymes that bind more metal ions. This selectivity is exemplified by using a series of in vitro selected DNAzymes that contain a single metal ligand modification at the cleavage junction.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1016/j.bios.2020.112285 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G20-17001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G20-17001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G20-17001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G20-17001/>Selection of a metal ligand modified DNAzyme for detecting Ni2+</a></strong><br><a href=/gwf-publications/people/W/Wei-Ren/ data-uni=uni0513>Wei Ren</a>,
<a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/D/Donatien-de-Rochambeau/ data-uni=uni0057>Donatien de Rochambeau</a>,
<a href=/gwf-publications/people/W/Woohyun-J-Moon/ data-uni=uni0354>Woohyun J. Moon</a>,
<a href=/gwf-publications/people/J/Jinyi-Zhang/ data-uni=uni1264>Jinyi Zhang</a>,
<a href=/gwf-publications/people/M/Mingsheng-Lyu/ data-uni=uni0836>Mingsheng Lyu</a>,
<a href=/gwf-publications/people/S/Shujun-Wang/ data-uni=uni0554>Shujun Wang</a>,
<a href=/gwf-publications/people/H/Hanadi-F-Sleiman/ data-uni=uni0057>Hanadi F. Sleiman</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G20-17/ class=text-muted>Biosensors and Bioelectronics, Volume 165</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G20-17001><div class="card-body p-3 small">Abstract Nickel is a highly important metal, and the detection of Ni2+ using biosensors is a long-stand analytical challenge. DNA has been widely used for metal detection, although no DNA-based sensors were reported for Ni2+. DNAzymes are DNA-based catalysts, and they recruit metal ions for catalysis. In this work, in vitro selection of RNA-cleaving DNAzymes was carried out using a library containing a region of 50 random nucleotides in the presence of Ni2+. To increase Ni2+ binding, a glycyl–histidine-functionalized tertiary amine moiety was inserted at the cleavage junction. A representative DNAzyme named Ni03 showed a high cleavage yield with Ni2+ and it was further studied. After truncation, the optimal sequence of Ni03l could bind one Ni2+ or two Co2+ for catalysis, while other metal ions were inactive. Its cleavage rates for 100 μM Ni2+ reached 0.63 h−1 at pH 8.0. A catalytic beacon biosensor was designed by labeling a fluorophore and a quencher on the Ni03l DNAzyme. Fluorescence enhancement was observed in the presence of Ni2+ with a detection limit of 12.9 μM. The sensor was also tested in spiked Lake Ontario water achieving a similar sensitivity. This is another example of using single-site modified DNAzyme for sensing transition metal ions.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1002/cbic.202000586 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G20-18001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G20-18001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G20-18001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G20-18001/>Zn &lt;sup>2+&lt;/sup> ‐Dependent DNAzymes: From Solution Chemistry to Analytical, Materials and Therapeutic Applications</a></strong><br><a href=/gwf-publications/people/W/Woohyun-J-Moon/ data-uni=uni0354>Woohyun J. Moon</a>,
<a href=/gwf-publications/people/Y/Yongjie-Yang/ data-uni=uni0926>Yongjie Yang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G20-18/ class=text-muted>ChemBioChem, Volume 22, Issue 5</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G20-18001><div class="card-body p-3 small">Since 1994, deoxyribozymes or DNAzymes have been in vitro selected to catalyze various types of reactions. Metal ions play a critical role in DNAzyme catalysis, and Zn2+ is a very important one among them. Zn2+ has good biocompatibility and can be used for intracellular applications. Chemically, Zn2+ is a Lewis acid and it can bind to both the phosphate backbone and the nucleobases of DNA. Zn2+ undergoes hydrolysis even at neutral pH, and the partially hydrolyzed polynuclear complexes can affect the interactions with DNA. These features have made Zn2+ a unique cofactor for DNAzyme reactions. This review summarizes Zn2+ -dependent DNAzymes with an emphasis on RNA-/DNA-cleaving reactions. A key feature is the sharp Zn2+ concentration and pH-dependent activity for many of the DNAzymes. The applications of these DNAzymes as biosensors for Zn2+ , as therapeutic agents to cleave intracellular RNA, and as chemical biology tools to manipulate DNA are discussed. Future studies can focus on the selection of new DNAzymes with improved performance and detailed biochemical characterizations to understand the role of Zn2+ , which can facilitate practical applications of Zn2+ -dependent DNAzymes.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1002/9780470027318.a9747 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G20-114001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G20-114001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G20-114001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G20-114001/>&lt;scp>DNA&lt;/scp> ‐Enabled Heavy Metal Detection in Water</a></strong><br><a href=/gwf-publications/people/W/Woohyun-J-Moon/ data-uni=uni0354>Woohyun J. Moon</a>,
<a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G20-114/ class=text-muted>Encyclopedia of Analytical Chemistry</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G20-114001><div class="card-body p-3 small">Detection of heavy metal contamination in the environment is an on-going analytical challenge. In effort of developing portable biosensors, deoxyribonucleic acid (DNA)-based designs have gained much attention for their high affinity and specificity to metals, stability, cost-efficiency, ease of modification, and batch-to-batch reproducibility. Specific sequences of DNA aptamers and DNAzymes provide grounds for rational designs of fluorescent, colorimetric, and electrochemical detection methods. Aptamers exert only a binding function, while DNAzymes can use heavy metals to catalyze specific chemical and biological transformations. This article starts with a brief introduction of heavy metals and their interactions with DNA. Then DNA aptamers and DNAzymes are respectively reviewed from their in vitro selection, representative DNA sequences, and design of biosensors. For signal transduction, various fluorescent, colorimetric, and electrochemical examples are described. Finally, future perspectives are discussed.</div></div><div data-prj=prj22><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1016/j.jhazmat.2019.121572 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G20-121001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G20-121001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G20-121001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G20-121001/>A DNA-based biosensor for aqueous Hg(II): Performance under variable pH, temperature and competing ligand composition</a></strong><br><a href=/gwf-publications/people/K/Kunfu-Pi/ data-uni=uni0174>Kunfu Pi</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a>,
<a href=/gwf-publications/people/P/Philippe-Van-Cappellen/ data-uni=uni0002>Philippe Van Cappellen</a><br><a href=/gwf-publications/volumes/G20-121/ class=text-muted>Journal of Hazardous Materials, Volume 385</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G20-121001><div class="card-body p-3 small">Abstract Mercury (Hg) is a toxic metal posing major health risks to human beings and wildlife. The characterization of Hg fate and transport in aquatic environments is hindered by a lack of sensitive, selective and easily field-deployable analytical techniques. Here we assess the reliability and performance of a Hg2+ sensor based on the selective binding of Hg2+ to a thymine-rich DNA under environmentally-relevant conditions. Experimental results indicate that the interactions between the DNA and SYBR Green I, which produce the detection fluorescence signal, are significantly impacted by pH, metal ligands and natural dissolved organic matter (NDOM). These interferences are largely eliminated by immobilizing the DNA in a polyacrylamide hydrogel, although high concentrations of NDOM, such as fulvic acids, still affect the sensor’s performance due to competitive binding of Hg2+. The binding of Hg2+ to NDOM, however, can be accounted for via equilibrium speciation calculations, which also yield the complexation constant for Hg2+ binding to the DNA in the hydrogel. The equilibrium calculations reproduce the results for the entire set of experimental conditions, from simple electrolyte solutions to complex aqueous compositions mimicking natural lake waters, and across large ranges of pH (3-10) and temperature (5-50 °C).</div></div><h4>2019</h4><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.3389/fchem.2019.00198 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G19-73001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G19-73001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G19-73001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G19-73001/>Phosphorothioate DNA Mediated Sequence-Insensitive Etching and Ripening of Silver Nanoparticles</a></strong><br><a href=/gwf-publications/people/S/Shengqiang-Hu/ data-uni=uni1331>Shengqiang Hu</a>,
<a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/J/Jianxiu-Wang/ data-uni=uni1332>Jianxiu Wang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G19-73/ class=text-muted>Frontiers in Chemistry, Volume 7</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G19-73001><div class="card-body p-3 small">Many DNA-functionalized nanomaterials and biosensors have been reported, but most have ignored the influence of DNA on the stability of nanoparticles. We observed that cytosine-rich DNA oligonucleotides can etch silver nanoparticles (AgNPs). In this work, we showed that phosphorothioate (PS)-modified DNA (PS-DNA) can etch AgNPs independently of DNA sequence, suggesting that the thio-modifications are playing the major role in etching. Compared to unmodified DNA (e.g., poly-cytosine DNA), the concentration of required PS DNA decreases sharply, and the reaction rate increases. Furthermore, etching by PS-DNA occurs quite independent of pH, which is also different from unmodified DNA. The PS-DNA mediated etching could also be controlled well by varying DNA length and conformation, and the number and location of PS modifications. With a higher activity of PS-DNA, the process of etching, ripening, and further etching was taken place sequentially. The etching ability is inhibited by forming duplex DNA and thus etching can be used to measure the concentration of complementary DNA.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1002/cbic.201900344 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G19-125001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G19-125001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G19-125001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G19-125001/>Replacing Mg&lt;sup>2+&lt;/sup>by Fe&lt;sup>2+&lt;/sup>for RNA‐Cleaving DNAzymes</a></strong><br><a href=/gwf-publications/people/W/Woohyun-J-Moon/ data-uni=uni0354>Woohyun J. Moon</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G19-125/ class=text-muted>ChemBioChem, Volume 21, Issue 3</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G19-125001><div class="card-body p-3 small">It has been proposed that Mg2+ and Fe2+ are very similar in interacting with ribozymes and some protein-based enzymes, but their activities with DNAzymes have yet to be studied. Here, the activity of Fe2+ as cofactor for a few RNA-cleaving DNAzymes is investigated. 17E is a well-studied DNAzyme that is active in the presence of many different divalent metal ions; it is highly active with Fe2+ with an apparent Kd of 29.7±2.3 μm and a kobs of 1.12±0.11 min-1 in the presence of 1 mm Fe2+ at pH 7.5. Fe2+ has 21-fold higher activity than Mg2+ . Six different DNAzymes are then tested, and only the DNAzymes active with Mg2+ (17E, 8-17, and E5) are active with Fe2+ . Fe2+ has 25 and one- to twofold higher activity than Mg2+ for the 8-17 and E5 DNAzymes, respectively. In pH>7 buffer and in presence of air, 1 mm Fe2+ results in a nonspecific degradation of the DNA strand due to reactive oxygen species (ROS). Cleavage reactions in anoxic environment and antioxidant ascorbate can be used to overcome the effect of oxidation. The findings provide insights for potential DNAzyme catalysis in the early Earth, and they further support the similarity between Mg2+ and Fe2+ in enzyme catalysis.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1021/acsanm.9b02356 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G19-174001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G19-174001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G19-174001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G19-174001/>Synergistically Boosted Degradation of Organic Dyes by CeO&lt;sub>2&lt;/sub> Nanoparticles with Fluoride at Low pH</a></strong><br><a href=/gwf-publications/people/Y/Yawen-Wang/ data-uni=uni0855>Yawen Wang</a>,
<a href=/gwf-publications/people/L/Lei-Zhu/ data-uni=uni1283>Lei Zhu</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G19-174/ class=text-muted>ACS Applied Nano Materials, Volume 3, Issue 1</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G19-174001><div class="card-body p-3 small">Catalysts that can work without the need of light and additional oxidants such as H2O2 to degrade organic pollutants have been long sought. In this work, we report that at acidic condition, CeO2 na...</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1039/c9cc06167c data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G19-175001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G19-175001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G19-175001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G19-175001/>Intentional hydrolysis to overcome the hydrolysis problem: detection of Ce(&lt;scp>iv&lt;/scp>) by producing oxidase-like nanozymes with F&lt;sup>−&lt;/sup></a></strong><br><a href=/gwf-publications/people/Y/Yawen-Wang/ data-uni=uni0855>Yawen Wang</a>,
<a href=/gwf-publications/people/J/Jianzhong-Yang/ data-uni=uni1284>Jianzhong Yang</a>,
<a href=/gwf-publications/people/Y/Yilin-Zhao/ data-uni=uni0004>Yilin Zhao</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G19-175/ class=text-muted>Chemical Communications, Volume 55, Issue 89</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G19-175001><div class="card-body p-3 small">Fluoride boosts the oxidase-like activity of hydrolyzed Ce(&lt;sc>iv&lt;/sc>) but inhibits the activity of Ce(&lt;sc>iv&lt;/sc>), allowing intentional hydrolysis to be performed for consistent analysis of Ce(&lt;sc>iv&lt;/sc>).</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1021/acs.analchem.9b02789 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G19-193001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G19-193001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G19-193001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G19-193001/>The Arsenic-Binding Aptamer Cannot Bind Arsenic: Critical Evaluation of Aptamer Selection and Binding</a></strong><br><a href=/gwf-publications/people/C/Chenghua-Zong/ data-uni=uni0353>Chenghua Zong</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G19-193/ class=text-muted>Analytical Chemistry, Volume 91, Issue 16</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G19-193001><div class="card-body p-3 small">An arsenic-binding aptamer named Ars-3 was reported in 2009, and it has been used for detection of As(III) in more than two dozen papers. In this work, we performed extensive binding assays using isothermal titration calorimetry, various DNA-staining dyes, and gold nanoparticles. By carefully comparing Ars-3 and a few random control DNA sequences, no specific binding of As(III) was observed in each case. Therefore, we conclude that Ars-3 cannot bind As(III). Possible reasons for some of the previously reported binding and detection were speculated to be related to the adsorption of As(III) onto gold surfaces, which were used in many related sensor designs, and As(III)/Au interactions were not considered before. The selection data in the original paper were then analyzed in terms of sequence alignment, secondary structure prediction, and dissociation constant measurement. These steps need rigorous testing before confirming specific binding of newly selected aptamers. This study calls for attention to the gap between aptamer selection and biosensor design, and the gap needs to be filled by careful binding assays to further the growth of the aptamer field.</div></div><h4>2018</h4><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1021/acsomega.8b02039 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G18-44001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G18-44001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G18-44001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G18-44001/>Reselection Yielding a Smaller and More Active Silver-Specific DNAzyme</a></strong><br><a href=/gwf-publications/people/L/Lide-Gu/ data-uni=uni1342>Lide Gu</a>,
<a href=/gwf-publications/people/R/Runjhun-Saran/ data-uni=uni0002>Runjhun Saran</a>,
<a href=/gwf-publications/people/W/Wanli-Yan/ data-uni=uni1343>Wanli Yan</a>,
<a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/S/Shujun-Wang/ data-uni=uni0554>Shujun Wang</a>,
<a href=/gwf-publications/people/M/Mingsheng-Lyu/ data-uni=uni0836>Mingsheng Lyu</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G18-44/ class=text-muted>ACS Omega, Volume 3, Issue 11</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G18-44001><div class="card-body p-3 small">Ag10c is a recently reported RNA-cleaving DNAzyme obtained from in vitro selection. Its cleavage activity selectively requires Ag+ ions, and thus it has been used as a sensor for Ag+ detection. However, the previous selection yielded very limited information regarding its sequence requirement, since only ∼0.1% of the population in the final library were related to Ag10c and most other sequences were inactive. In this work, we performed a reselection by randomizing the 19 important nucleotides in Ag10c in such a way that a purine has an equal chance of being A or G, whereas a pyrimidine has an equal chance of being T or C. The round 3 library of the reselection was carefully analyzed and a statistic understanding of the relative importance of each nucleotide was obtained. At the same time, a more active mutant was identified, containing two mutated nucleotides. Further analysis indicated new base pairs leading to an enzyme with smaller catalytic loops but with ∼200% activity of the original Ag10c, and also excellent selectivity for Ag+. Therefore, a more active mutant of Ag10c was obtained and further truncations were successfully performed, which might be better candidates for developing new biosensors for silver. A deeper biochemical understanding was also obtained using this reselection method.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1093/nar/gky807 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G18-52001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G18-52001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G18-52001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G18-52001/>Misfolding of a DNAzyme for ultrahigh sodium selectivity over potassium</a></strong><br><a href=/gwf-publications/people/Y/Yanping-He/ data-uni=uni0394>Yanping He</a>,
<a href=/gwf-publications/people/D/Da-Chen/ data-uni=uni1357>Da Chen</a>,
<a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/Y/Yibo-Zhou/ data-uni=uni1358>Yibo Zhou</a>,
<a href=/gwf-publications/people/L/Lingzi-Ma/ data-uni=uni0002>Lingzi Ma</a>,
<a href=/gwf-publications/people/K/Kexin-Xu/ data-uni=uni0482>Kexin Xu</a>,
<a href=/gwf-publications/people/R/Ronghua-Yang/ data-uni=uni1359>Ronghua Yang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G18-52/ class=text-muted>Nucleic Acids Research, Volume 46, Issue 19</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G18-52001><div class="card-body p-3 small">Herein, the excellent Na+ selectivity of a few RNA-cleaving DNAzymes was exploited, where Na+ can be around 3000-fold more effective than K+ for promoting catalysis. By using a double mutant based on the Ce13d DNAzyme, and by lowering the temperature, increased 2-aminopurine (2AP) fluorescence was observed with addition of both Na+ and K+. The fluorescence increase was similar for these two metals at below 10 mM, after which K+ took a different pathway. Since 2AP probes its local base stacking environment, K+ can be considered to induce misfolding. Binding of both Na+ and K+ was specific, since single base mutations could fully inhibit 2AP fluorescence for both metals. The binding thermodynamics was measured by temperature-dependent experiments revealing enthalpy-driven binding for both metals and less coordination sites compared to G-quadruplex DNA. Cleavage activity assays indicated a moderate cleavage activity with 10 mM K+, while further increase of K+ inhibited the activity, also supporting its misfolding of the DNAzyme. For comparison, a G-quadruplex DNA was also studied using the same system, where Na+ and K+ led to the same final state with only around 8-fold difference in Kd. This study provides interesting insights into strategies for discriminating Na+ and K+.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1021/acs.biochem.8b00900 data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G18-57001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G18-57001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G18-57001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G18-57001/>Instantaneous Iodine-Assisted DNAzyme Cleavage of Phosphorothioate RNA</a></strong><br><a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/W/Woohyun-J-Moon/ data-uni=uni0354>Woohyun J. Moon</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G18-57/ class=text-muted>Biochemistry, Volume 58, Issue 5</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G18-57001><div class="card-body p-3 small">Metal ions play a critical role in the RNA-cleavage reaction by interacting with the scissile phosphate and stabilizing the highly negatively charged transition state. Many metal-dependent DNAzymes have been selected for RNA cleavage. Herein, we report that the Ce13d DNAzyme can use nonmetallic iodine (I2) to cleave a phosphorothioate (PS)-modified substrate. The cleavage yield exceeded 60% for both the Rp and Sp stereoisomers in 10 s, while the yield without the enzyme strand was only ∼10%. The Ce13d cleavage with I2 also required Na+, consistent with the property of Ce13d and confirming the similar role of I2 as a metal ion. Ce13d had the highest yield among eight tested DNAzymes, with the second highest DNAzyme showing only 20% cleavage. The incomplete cleavage was due to competition from desulfurization and isomerization reactions. This DNAzyme was engineered for fluorescence-based I2 detection. With EDTA for masking metal ions, I2 was selectively detected down to 4.7 nM. Oxidation of I- with Fe3+ produced I2 in situ, allowing detection of Fe3+ down to 78 nM. By harnessing nonelectrostatic interactions, such as the I2/sulfur interaction observed here, more nonmetal species might be discovered to assist DNAzyme-based RNA cleavage.</div></div><div data-prj=prj38><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://www.doi.org/10.1039/c8cc07062h data-toggle=tooltip data-placement=top title="Publisher's Website" target=_blank>DOI
</a><a class="badge badge-secondary align-middle mr-1" href=/gwf-publications/G18-66001.bib data-toggle=tooltip data-placement=top title="Export to BibTeX">bib
</a><a class="badge badge-info align-middle mr-1" href=#abstract-G18-66001 data-toggle=collapse aria-expanded=false aria-controls=abstract-G18-66001 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/gwf-publications/G18-66001/>Highly active fluorogenic oxidase-mimicking NiO nanozymes</a></strong><br><a href=/gwf-publications/people/L/Li-Dai/ data-uni=uni1376>Li Dai</a>,
<a href=/gwf-publications/people/B/Biwu-Liu/ data-uni=uni1377>Biwu Liu</a>,
<a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ data-uni=uni0002>Po‐Jung Jimmy Huang</a>,
<a href=/gwf-publications/people/Z/Zijie-Zhang/ data-uni=uni1346>Zijie Zhang</a>,
<a href=/gwf-publications/people/J/Juewen-Liu/ data-uni=uni0002>Juewen Liu</a><br><a href=/gwf-publications/volumes/G18-66/ class=text-muted>Chemical Communications, Volume 54, Issue 88</a></span></p></div><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-G18-66001><div class="card-body p-3 small">NiO nanoparticles can quickly catalyze oxidation of Amplex red to produce fluorescent products for intracellular imaging, much more efficiently than other types of tested nanozymes.</div></div></div><div class=col-lg-3><a class="btn btn-lg btn-secondary btn-block mb-2" href="https://www.semanticscholar.org/search?q=Juewen+Liu" title="Search for 'Juewen Liu' on Semantic Scholar"><i class="ai ai-semantic-scholar"></i><span class=pl-sm-2>Search</span></a><div class=row><div class="col-12 col-md-6 col-lg-12"><div class=card><h5 class=card-header>Co-authors</h5><ul class="list-group list-group-flush list-group-compact"><li class=list-group-item><a href=/gwf-publications/people/P/PoJung-Jimmy-Huang/ class=align-middle data-uni=uni0002>Po‐Jung Jimmy Huang</a>
<span class="badge badge-secondary align-middle ml-2">18</span></li><li class=list-group-item><a href=/gwf-publications/people/W/Woohyun-J-Moon/ class=align-middle data-uni=uni0354>Woohyun J. Moon</a>
<span class="badge badge-secondary align-middle ml-2">9</span></li><li class=list-group-item><a href=/gwf-publications/people/Z/Zhicheng-Huang/ class=align-middle data-uni=uni0272>Zhicheng Huang</a>
<span class="badge badge-secondary align-middle ml-2">8</span></li><li class=list-group-item><a href=/gwf-publications/people/P/Philippe-Van-Cappellen/ class=align-middle data-uni=uni0002>Philippe Van Cappellen</a>
<span class="badge badge-secondary align-middle ml-2">6</span></li><li class=list-group-item><a href=/gwf-publications/people/C/Chenghua-Zong/ class=align-middle data-uni=uni0353>Chenghua Zong</a>
<span class="badge badge-secondary align-middle ml-2">5</span></li><li class="list-group-item list-group-toggle-btn py-1" data-toggle=collapse data-target=#more-coauthors aria-expanded=false aria-controls=more-coauthors>show all...</li><div class="collapse border-top" id=more-coauthors><li class=list-group-item><a href=/gwf-publications/people/L/Lingzi-Ma/ class=align-middle data-uni=uni0002>Lingzi Ma</a>
<span class="badge badge-secondary align-middle ml-2">5</span></li><li class=list-group-item><a href=/gwf-publications/people/R/Runjhun-Saran/ class=align-middle data-uni=uni0002>Runjhun Saran</a>
<span class="badge badge-secondary align-middle ml-2">5</span></li><li class=list-group-item><a href=/gwf-publications/people/X/Xiaoting-Jin/ class=align-middle data-uni=uni0831>Xiaoting Jin</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Mohamad-Zandieh/ class=align-middle data-uni=uni0002>Mohamad Zandieh</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/E/Erin-Griffiths/ class=align-middle data-uni=uni0002>Erin Griffiths</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/A/Alexander-Waldie/ class=align-middle data-uni=uni0002>Alexander Waldie</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Shuhuan-Li/ class=align-middle data-uni=uni0161>Shuhuan Li</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/J/John-F-Honek/ class=align-middle data-uni=uni0002>John F. Honek</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/F/Fereidoun-Rezanezhad/ class=align-middle data-uni=uni0002>Fereidoun Rezanezhad</a>
<span class="badge badge-secondary align-middle ml-2">4</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yongjie-Yang/ class=align-middle data-uni=uni0926>Yongjie Yang</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yichen-Zhao/ class=align-middle data-uni=uni0002>Yichen Zhao</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/K/Kayvan-Yavari/ class=align-middle data-uni=uni0002>Kayvan Yavari</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/K/Kunfu-Pi/ class=align-middle data-uni=uni0174>Kunfu Pi</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/D/Donatien-de-Rochambeau/ class=align-middle data-uni=uni0057>Donatien de Rochambeau</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/H/Hanadi-F-Sleiman/ class=align-middle data-uni=uni0057>Hanadi F. Sleiman</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/M/Mingsheng-Lyu/ class=align-middle data-uni=uni0836>Mingsheng Lyu</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Shujun-Wang/ class=align-middle data-uni=uni0554>Shujun Wang</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yawen-Wang/ class=align-middle data-uni=uni0855>Yawen Wang</a>
<span class="badge badge-secondary align-middle ml-2">2</span></li><li class=list-group-item><a href=/gwf-publications/people/W/Weixuan-Li/ class=align-middle data-uni=uni0663>Weixuan Li</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yihao-Wang/ class=align-middle data-uni=uni0002>Yihao Wang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/W/Wei-Ren/ class=align-middle data-uni=uni0513>Wei Ren</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jinyi-Zhang/ class=align-middle data-uni=uni1264>Jinyi Zhang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/L/Lide-Gu/ class=align-middle data-uni=uni1342>Lide Gu</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/W/Wanli-Yan/ class=align-middle data-uni=uni1343>Wanli Yan</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yanping-He/ class=align-middle data-uni=uni0394>Yanping He</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/D/Da-Chen/ class=align-middle data-uni=uni1357>Da Chen</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yibo-Zhou/ class=align-middle data-uni=uni1358>Yibo Zhou</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/K/Kexin-Xu/ class=align-middle data-uni=uni0482>Kexin Xu</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/R/Ronghua-Yang/ class=align-middle data-uni=uni1359>Ronghua Yang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/L/Li-Dai/ class=align-middle data-uni=uni1376>Li Dai</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/B/Biwu-Liu/ class=align-middle data-uni=uni1377>Biwu Liu</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Z/Zijie-Zhang/ class=align-middle data-uni=uni1346>Zijie Zhang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/S/Shengqiang-Hu/ class=align-middle data-uni=uni1331>Shengqiang Hu</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jianxiu-Wang/ class=align-middle data-uni=uni1332>Jianxiu Wang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/L/Lei-Zhu/ class=align-middle data-uni=uni1283>Lei Zhu</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/J/Jianzhong-Yang/ class=align-middle data-uni=uni1284>Jianzhong Yang</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li><li class=list-group-item><a href=/gwf-publications/people/Y/Yilin-Zhao/ class=align-middle data-uni=uni0004>Yilin Zhao</a>
<span class="badge badge-secondary align-middle ml-2">1</span></li></div></ul></div></div><div class="col-12 col-md-6 col-lg-12"><div class="card my-2 my-md-0 my-lg-2"><h5 class=card-header>Venues</h5><ul class="list-group list-group-flush list-group-compact"><li class=list-group-item><a href=/gwf-publications/venues/gwf/ class=align-middle>GWF</a><span class="badge badge-secondary align-middle ml-2">31</span></li></ul></div></div></div></div></div></section></div><footer class="bg-gradient-light py-2 py-xl-3 mt-3 mt-md-4 mt-xl-5"><div class=container><p class="text-muted small px-1">Global Water Futures Publications!</p></div></footer><script src=https://code.jquery.com/jquery-3.3.1.slim.min.js integrity=sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js integrity=sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut crossorigin=anonymous></script><script src=https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js integrity=sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k crossorigin=anonymous></script><script>$(function(){$('[data-toggle="tooltip"]').tooltip(),$("#toggle-all-abstracts")&&($("#toggle-all-abstracts").click(function(){var e=$("#toggle-all-abstracts");e.attr("disabled",!0),e.attr("data-toggle-state")=="hide"?($(".abstract-collapse").collapse("show"),e.attr("data-toggle-state","show")):($(".abstract-collapse").collapse("hide"),e.attr("data-toggle-state","hide")),e.attr("disabled",!1)}),$("#toggle-all-abstracts").attr("disabled",!1))})</script></body></html>